[2021-04-27 00:00:06.926 DEBUG     yangsuite.ysnetconf.rpcbuilder : 402]
  ...minus pfx/localname: /mdt-cfg:mdt-subscription[subscription-id="105"]/mdt-cfg:base/mdt-cfg:source-address
[2021-04-27 00:00:06.942 DEBUG     yangsuite.ysnetconf.rpcbuilder : 378]
Parsing remnant of xpath: /mdt-cfg:mdt-subscription[subscription-id="105"]/mdt-cfg:base/mdt-cfg:source-address
[2021-04-27 00:00:06.943 DEBUG     yangsuite.ysnetconf.rpcbuilder : 402]
  ...minus pfx/localname: [subscription-id="105"]/mdt-cfg:base/mdt-cfg:source-address
[2021-04-27 00:00:06.943 DEBUG     yangsuite.ysnetconf.rpcbuilder : 433]
  ...minus keys / values: /mdt-cfg:base/mdt-cfg:source-address
[2021-04-27 00:00:06.944 DEBUG     yangsuite.ysnetconf.rpcbuilder : 378]
Parsing remnant of xpath: /mdt-cfg:base/mdt-cfg:source-address
[2021-04-27 00:00:06.945 DEBUG     yangsuite.ysnetconf.rpcbuilder : 402]
  ...minus pfx/localname: /mdt-cfg:source-address
[2021-04-27 00:00:06.945 DEBUG     yangsuite.ysnetconf.rpcbuilder : 378]
Parsing remnant of xpath: /mdt-cfg:source-address
[2021-04-27 00:00:06.946 DEBUG     yangsuite.ysnetconf.rpcbuilder : 402]
  ...minus pfx/localname: 
[2021-04-27 00:00:06.946 DEBUG     yangsuite.ysnetconf.rpcbuilder : 378]
Parsing remnant of xpath: /mdt-cfg:mdt-config-data/mdt-cfg:mdt-subscription[subscription-id="105"]/mdt-cfg:base/mdt-cfg:no-synch-on-start
[2021-04-27 00:00:06.947 DEBUG     yangsuite.ysnetconf.rpcbuilder : 402]
  ...minus pfx/localname: /mdt-cfg:mdt-subscription[subscription-id="105"]/mdt-cfg:base/mdt-cfg:no-synch-on-start
[2021-04-27 00:00:06.947 DEBUG     yangsuite.ysnetconf.rpcbuilder : 378]
Parsing remnant of xpath: /mdt-cfg:mdt-subscription[subscription-id="105"]/mdt-cfg:base/mdt-cfg:no-synch-on-start
[2021-04-27 00:00:06.948 DEBUG     yangsuite.ysnetconf.rpcbuilder : 402]
  ...minus pfx/localname: [subscription-id="105"]/mdt-cfg:base/mdt-cfg:no-synch-on-start
[2021-04-27 00:00:06.949 DEBUG     yangsuite.ysnetconf.rpcbuilder : 433]
  ...minus keys / values: /mdt-cfg:base/mdt-cfg:no-synch-on-start
[2021-04-27 00:00:06.950 DEBUG     yangsuite.ysnetconf.rpcbuilder : 378]
Parsing remnant of xpath: /mdt-cfg:base/mdt-cfg:no-synch-on-start
[2021-04-27 00:00:06.950 DEBUG     yangsuite.ysnetconf.rpcbuilder : 402]
  ...minus pfx/localname: /mdt-cfg:no-synch-on-start
[2021-04-27 00:00:06.951 DEBUG     yangsuite.ysnetconf.rpcbuilder : 378]
Parsing remnant of xpath: /mdt-cfg:no-synch-on-start
[2021-04-27 00:00:06.953 DEBUG     yangsuite.ysnetconf.rpcbuilder : 402]
  ...minus pfx/localname: 
[2021-04-27 00:00:06.953 DEBUG     yangsuite.ysnetconf.rpcbuilder : 378]
Parsing remnant of xpath: /mdt-cfg:mdt-config-data/mdt-cfg:mdt-subscription[subscription-id="105"]/mdt-cfg:base/mdt-cfg:no-synch-on-start
[2021-04-27 00:00:06.954 DEBUG     yangsuite.ysnetconf.rpcbuilder : 402]
  ...minus pfx/localname: /mdt-cfg:mdt-subscription[subscription-id="105"]/mdt-cfg:base/mdt-cfg:no-synch-on-start
[2021-04-27 00:00:06.955 DEBUG     yangsuite.ysnetconf.rpcbuilder : 378]
Parsing remnant of xpath: /mdt-cfg:mdt-subscription[subscription-id="105"]/mdt-cfg:base/mdt-cfg:no-synch-on-start
[2021-04-27 00:00:06.955 DEBUG     yangsuite.ysnetconf.rpcbuilder : 402]
  ...minus pfx/localname: [subscription-id="105"]/mdt-cfg:base/mdt-cfg:no-synch-on-start
[2021-04-27 00:00:06.955 DEBUG     yangsuite.ysnetconf.rpcbuilder : 433]
  ...minus keys / values: /mdt-cfg:base/mdt-cfg:no-synch-on-start
[2021-04-27 00:00:06.956 DEBUG     yangsuite.ysnetconf.rpcbuilder : 378]
Parsing remnant of xpath: /mdt-cfg:base/mdt-cfg:no-synch-on-start
[2021-04-27 00:00:06.960 DEBUG     yangsuite.ysnetconf.rpcbuilder : 402]
  ...minus pfx/localname: /mdt-cfg:no-synch-on-start
[2021-04-27 00:00:06.960 DEBUG     yangsuite.ysnetconf.rpcbuilder : 378]
Parsing remnant of xpath: /mdt-cfg:no-synch-on-start
[2021-04-27 00:00:06.961 DEBUG     yangsuite.ysnetconf.rpcbuilder : 402]
  ...minus pfx/localname: 
[2021-04-27 00:00:06.961 DEBUG     yangsuite.ysnetconf.rpcbuilder : 378]
Parsing remnant of xpath: /mdt-cfg:mdt-config-data/mdt-cfg:mdt-subscription[subscription-id="105"]/mdt-cfg:base/mdt-cfg:xpath
[2021-04-27 00:00:06.962 DEBUG     yangsuite.ysnetconf.rpcbuilder : 402]
  ...minus pfx/localname: /mdt-cfg:mdt-subscription[subscription-id="105"]/mdt-cfg:base/mdt-cfg:xpath
[2021-04-27 00:00:06.962 DEBUG     yangsuite.ysnetconf.rpcbuilder : 378]
Parsing remnant of xpath: /mdt-cfg:mdt-subscription[subscription-id="105"]/mdt-cfg:base/mdt-cfg:xpath
[2021-04-27 00:00:06.963 DEBUG     yangsuite.ysnetconf.rpcbuilder : 402]
  ...minus pfx/localname: [subscription-id="105"]/mdt-cfg:base/mdt-cfg:xpath
[2021-04-27 00:00:06.964 DEBUG     yangsuite.ysnetconf.rpcbuilder : 433]
  ...minus keys / values: /mdt-cfg:base/mdt-cfg:xpath
[2021-04-27 00:00:06.964 DEBUG     yangsuite.ysnetconf.rpcbuilder : 378]
Parsing remnant of xpath: /mdt-cfg:base/mdt-cfg:xpath
[2021-04-27 00:00:06.965 DEBUG     yangsuite.ysnetconf.rpcbuilder : 402]
  ...minus pfx/localname: /mdt-cfg:xpath
[2021-04-27 00:00:06.965 DEBUG     yangsuite.ysnetconf.rpcbuilder : 378]
Parsing remnant of xpath: /mdt-cfg:xpath
[2021-04-27 00:00:06.965 DEBUG     yangsuite.ysnetconf.rpcbuilder : 402]
  ...minus pfx/localname: 
[2021-04-27 00:00:06.966 DEBUG     yangsuite.ysnetconf.rpcbuilder : 378]
Parsing remnant of xpath: /mdt-cfg:mdt-config-data/mdt-cfg:mdt-subscription[subscription-id="105"]/mdt-cfg:base/mdt-cfg:xpath
[2021-04-27 00:00:06.966 DEBUG     yangsuite.ysnetconf.rpcbuilder : 402]
  ...minus pfx/localname: /mdt-cfg:mdt-subscription[subscription-id="105"]/mdt-cfg:base/mdt-cfg:xpath
[2021-04-27 00:00:06.966 DEBUG     yangsuite.ysnetconf.rpcbuilder : 378]
Parsing remnant of xpath: /mdt-cfg:mdt-subscription[subscription-id="105"]/mdt-cfg:base/mdt-cfg:xpath
[2021-04-27 00:00:06.967 DEBUG     yangsuite.ysnetconf.rpcbuilder : 402]
  ...minus pfx/localname: [subscription-id="105"]/mdt-cfg:base/mdt-cfg:xpath
[2021-04-27 00:00:06.967 DEBUG     yangsuite.ysnetconf.rpcbuilder : 433]
  ...minus keys / values: /mdt-cfg:base/mdt-cfg:xpath
[2021-04-27 00:00:06.968 DEBUG     yangsuite.ysnetconf.rpcbuilder : 378]
Parsing remnant of xpath: /mdt-cfg:base/mdt-cfg:xpath
[2021-04-27 00:00:06.968 DEBUG     yangsuite.ysnetconf.rpcbuilder : 402]
  ...minus pfx/localname: /mdt-cfg:xpath
[2021-04-27 00:00:06.969 DEBUG     yangsuite.ysnetconf.rpcbuilder : 378]
Parsing remnant of xpath: /mdt-cfg:xpath
[2021-04-27 00:00:06.970 DEBUG     yangsuite.ysnetconf.rpcbuilder : 402]
  ...minus pfx/localname: 
[2021-04-27 00:00:06.970 DEBUG     yangsuite.ysnetconf.rpcbuilder : 378]
Parsing remnant of xpath: /mdt-cfg:mdt-config-data/mdt-cfg:mdt-subscription[subscription-id="105"]/mdt-cfg:mdt-receivers[address="192.168.42.128"][port="57000"]/mdt-cfg:protocol
[2021-04-27 00:00:06.971 DEBUG     yangsuite.ysnetconf.rpcbuilder : 402]
  ...minus pfx/localname: /mdt-cfg:mdt-subscription[subscription-id="105"]/mdt-cfg:mdt-receivers[address="192.168.42.128"][port="57000"]/mdt-cfg:protocol
[2021-04-27 00:00:06.971 DEBUG     yangsuite.ysnetconf.rpcbuilder : 378]
Parsing remnant of xpath: /mdt-cfg:mdt-subscription[subscription-id="105"]/mdt-cfg:mdt-receivers[address="192.168.42.128"][port="57000"]/mdt-cfg:protocol
[2021-04-27 00:00:06.971 DEBUG     yangsuite.ysnetconf.rpcbuilder : 402]
  ...minus pfx/localname: [subscription-id="105"]/mdt-cfg:mdt-receivers[address="192.168.42.128"][port="57000"]/mdt-cfg:protocol
[2021-04-27 00:00:06.972 DEBUG     yangsuite.ysnetconf.rpcbuilder : 433]
  ...minus keys / values: /mdt-cfg:mdt-receivers[address="192.168.42.128"][port="57000"]/mdt-cfg:protocol
[2021-04-27 00:00:06.973 DEBUG     yangsuite.ysnetconf.rpcbuilder : 378]
Parsing remnant of xpath: /mdt-cfg:mdt-receivers[address="192.168.42.128"][port="57000"]/mdt-cfg:protocol
[2021-04-27 00:00:06.973 DEBUG     yangsuite.ysnetconf.rpcbuilder : 402]
  ...minus pfx/localname: [address="192.168.42.128"][port="57000"]/mdt-cfg:protocol
[2021-04-27 00:00:06.973 DEBUG     yangsuite.ysnetconf.rpcbuilder : 433]
  ...minus keys / values: [port="57000"]/mdt-cfg:protocol
[2021-04-27 00:00:06.974 DEBUG     yangsuite.ysnetconf.rpcbuilder : 433]
  ...minus keys / values: /mdt-cfg:protocol
[2021-04-27 00:00:06.974 DEBUG     yangsuite.ysnetconf.rpcbuilder : 378]
Parsing remnant of xpath: /mdt-cfg:protocol
[2021-04-27 00:00:06.975 DEBUG     yangsuite.ysnetconf.rpcbuilder : 402]
  ...minus pfx/localname: 
[2021-04-27 00:00:06.975 DEBUG     yangsuite.ysnetconf.rpcbuilder : 378]
Parsing remnant of xpath: /mdt-cfg:mdt-config-data/mdt-cfg:mdt-subscription[subscription-id="105"]/mdt-cfg:mdt-receivers[address="192.168.42.128"][port="57000"]/mdt-cfg:protocol
[2021-04-27 00:00:06.976 DEBUG     yangsuite.ysnetconf.rpcbuilder : 402]
  ...minus pfx/localname: /mdt-cfg:mdt-subscription[subscription-id="105"]/mdt-cfg:mdt-receivers[address="192.168.42.128"][port="57000"]/mdt-cfg:protocol
[2021-04-27 00:00:06.976 DEBUG     yangsuite.ysnetconf.rpcbuilder : 378]
Parsing remnant of xpath: /mdt-cfg:mdt-subscription[subscription-id="105"]/mdt-cfg:mdt-receivers[address="192.168.42.128"][port="57000"]/mdt-cfg:protocol
[2021-04-27 00:00:06.977 DEBUG     yangsuite.ysnetconf.rpcbuilder : 402]
  ...minus pfx/localname: [subscription-id="105"]/mdt-cfg:mdt-receivers[address="192.168.42.128"][port="57000"]/mdt-cfg:protocol
[2021-04-27 00:00:06.977 DEBUG     yangsuite.ysnetconf.rpcbuilder : 433]
  ...minus keys / values: /mdt-cfg:mdt-receivers[address="192.168.42.128"][port="57000"]/mdt-cfg:protocol
[2021-04-27 00:00:06.978 DEBUG     yangsuite.ysnetconf.rpcbuilder : 378]
Parsing remnant of xpath: /mdt-cfg:mdt-receivers[address="192.168.42.128"][port="57000"]/mdt-cfg:protocol
[2021-04-27 00:00:06.979 DEBUG     yangsuite.ysnetconf.rpcbuilder : 402]
  ...minus pfx/localname: [address="192.168.42.128"][port="57000"]/mdt-cfg:protocol
[2021-04-27 00:00:06.979 DEBUG     yangsuite.ysnetconf.rpcbuilder : 433]
  ...minus keys / values: [port="57000"]/mdt-cfg:protocol
[2021-04-27 00:00:06.979 DEBUG     yangsuite.ysnetconf.rpcbuilder : 433]
  ...minus keys / values: /mdt-cfg:protocol
[2021-04-27 00:00:06.980 DEBUG     yangsuite.ysnetconf.rpcbuilder : 378]
Parsing remnant of xpath: /mdt-cfg:protocol
[2021-04-27 00:00:06.980 DEBUG     yangsuite.ysnetconf.rpcbuilder : 402]
  ...minus pfx/localname: 
[2021-04-27 00:00:06.981 DEBUG     yangsuite.ysnetconf.rpcbuilder : 262]
get_payload: constructed XML:
<filter xmlns="urn:ietf:params:xml:ns:netconf:base:1.0">
  <mdt-config-data xmlns="http://cisco.com/ns/yang/Cisco-IOS-XE-mdt-cfg">
    <mdt-subscription>
      <subscription-id>105</subscription-id>
      <base>
        <stream/>
        <encoding>encode-kvgpb</encoding>
        <source-address>192.168.42.82</source-address>
        <no-synch-on-start>true</no-synch-on-start>
        <xpath>/if:interfaces/interface/enabled</xpath>
      </base>
      <mdt-receivers>
        <address>192.168.42.128</address>
        <port>57000</port>
        <protocol>grpc-tcp</protocol>
      </mdt-receivers>
    </mdt-subscription>
  </mdt-config-data>
</filter>

[2021-04-27 00:00:06.984 INFO                       django.server : 124]
"POST /netconf/getrpc/ HTTP/1.1" 200 882
[2021-04-27 00:00:20.809 DEBUG          yangsuite.ysnetconf.nconf :1234]
Constructing raw RPC for edit-config
[2021-04-27 00:00:20.810 DEBUG     yangsuite.ysnetconf.rpcbuilder : 378]
Parsing remnant of xpath: /mdt-cfg:mdt-config-data/mdt-cfg:mdt-subscription[subscription-id="105"]/mdt-cfg:base/mdt-cfg:stream
[2021-04-27 00:00:20.811 DEBUG     yangsuite.ysnetconf.rpcbuilder : 402]
  ...minus pfx/localname: /mdt-cfg:mdt-subscription[subscription-id="105"]/mdt-cfg:base/mdt-cfg:stream
[2021-04-27 00:00:20.812 DEBUG     yangsuite.ysnetconf.rpcbuilder : 378]
Parsing remnant of xpath: /mdt-cfg:mdt-subscription[subscription-id="105"]/mdt-cfg:base/mdt-cfg:stream
[2021-04-27 00:00:20.812 DEBUG     yangsuite.ysnetconf.rpcbuilder : 402]
  ...minus pfx/localname: [subscription-id="105"]/mdt-cfg:base/mdt-cfg:stream
[2021-04-27 00:00:20.813 DEBUG     yangsuite.ysnetconf.rpcbuilder : 433]
  ...minus keys / values: /mdt-cfg:base/mdt-cfg:stream
[2021-04-27 00:00:20.814 DEBUG     yangsuite.ysnetconf.rpcbuilder : 378]
Parsing remnant of xpath: /mdt-cfg:base/mdt-cfg:stream
[2021-04-27 00:00:20.814 DEBUG     yangsuite.ysnetconf.rpcbuilder : 402]
  ...minus pfx/localname: /mdt-cfg:stream
[2021-04-27 00:00:20.815 DEBUG     yangsuite.ysnetconf.rpcbuilder : 378]
Parsing remnant of xpath: /mdt-cfg:stream
[2021-04-27 00:00:20.815 DEBUG     yangsuite.ysnetconf.rpcbuilder : 402]
  ...minus pfx/localname: 
[2021-04-27 00:00:20.816 DEBUG     yangsuite.ysnetconf.rpcbuilder : 378]
Parsing remnant of xpath: /mdt-cfg:mdt-config-data/mdt-cfg:mdt-subscription[subscription-id="105"]/mdt-cfg:base/mdt-cfg:stream
[2021-04-27 00:00:20.817 DEBUG     yangsuite.ysnetconf.rpcbuilder : 402]
  ...minus pfx/localname: /mdt-cfg:mdt-subscription[subscription-id="105"]/mdt-cfg:base/mdt-cfg:stream
[2021-04-27 00:00:20.817 DEBUG     yangsuite.ysnetconf.rpcbuilder : 378]
Parsing remnant of xpath: /mdt-cfg:mdt-subscription[subscription-id="105"]/mdt-cfg:base/mdt-cfg:stream
[2021-04-27 00:00:20.818 DEBUG     yangsuite.ysnetconf.rpcbuilder : 402]
  ...minus pfx/localname: [subscription-id="105"]/mdt-cfg:base/mdt-cfg:stream
[2021-04-27 00:00:20.819 DEBUG     yangsuite.ysnetconf.rpcbuilder : 433]
  ...minus keys / values: /mdt-cfg:base/mdt-cfg:stream
[2021-04-27 00:00:20.821 DEBUG     yangsuite.ysnetconf.rpcbuilder : 378]
Parsing remnant of xpath: /mdt-cfg:base/mdt-cfg:stream
[2021-04-27 00:00:20.821 DEBUG     yangsuite.ysnetconf.rpcbuilder : 402]
  ...minus pfx/localname: /mdt-cfg:stream
[2021-04-27 00:00:20.822 DEBUG     yangsuite.ysnetconf.rpcbuilder : 378]
Parsing remnant of xpath: /mdt-cfg:stream
[2021-04-27 00:00:20.823 DEBUG     yangsuite.ysnetconf.rpcbuilder : 402]
  ...minus pfx/localname: 
[2021-04-27 00:00:20.824 DEBUG     yangsuite.ysnetconf.rpcbuilder : 378]
Parsing remnant of xpath: /mdt-cfg:mdt-config-data/mdt-cfg:mdt-subscription[subscription-id="105"]/mdt-cfg:base/mdt-cfg:encoding
[2021-04-27 00:00:20.825 DEBUG     yangsuite.ysnetconf.rpcbuilder : 402]
  ...minus pfx/localname: /mdt-cfg:mdt-subscription[subscription-id="105"]/mdt-cfg:base/mdt-cfg:encoding
[2021-04-27 00:00:20.826 DEBUG     yangsuite.ysnetconf.rpcbuilder : 378]
Parsing remnant of xpath: /mdt-cfg:mdt-subscription[subscription-id="105"]/mdt-cfg:base/mdt-cfg:encoding
[2021-04-27 00:00:20.826 DEBUG     yangsuite.ysnetconf.rpcbuilder : 402]
  ...minus pfx/localname: [subscription-id="105"]/mdt-cfg:base/mdt-cfg:encoding
[2021-04-27 00:00:20.827 DEBUG     yangsuite.ysnetconf.rpcbuilder : 433]
  ...minus keys / values: /mdt-cfg:base/mdt-cfg:encoding
[2021-04-27 00:00:20.827 DEBUG     yangsuite.ysnetconf.rpcbuilder : 378]
Parsing remnant of xpath: /mdt-cfg:base/mdt-cfg:encoding
[2021-04-27 00:00:20.828 DEBUG     yangsuite.ysnetconf.rpcbuilder : 402]
  ...minus pfx/localname: /mdt-cfg:encoding
[2021-04-27 00:00:20.828 DEBUG     yangsuite.ysnetconf.rpcbuilder : 378]
Parsing remnant of xpath: /mdt-cfg:encoding
[2021-04-27 00:00:20.829 DEBUG     yangsuite.ysnetconf.rpcbuilder : 402]
  ...minus pfx/localname: 
[2021-04-27 00:00:20.829 DEBUG     yangsuite.ysnetconf.rpcbuilder : 378]
Parsing remnant of xpath: /mdt-cfg:mdt-config-data/mdt-cfg:mdt-subscription[subscription-id="105"]/mdt-cfg:base/mdt-cfg:encoding
[2021-04-27 00:00:20.830 DEBUG     yangsuite.ysnetconf.rpcbuilder : 402]
  ...minus pfx/localname: /mdt-cfg:mdt-subscription[subscription-id="105"]/mdt-cfg:base/mdt-cfg:encoding
[2021-04-27 00:00:20.830 DEBUG     yangsuite.ysnetconf.rpcbuilder : 378]
Parsing remnant of xpath: /mdt-cfg:mdt-subscription[subscription-id="105"]/mdt-cfg:base/mdt-cfg:encoding
[2021-04-27 00:00:20.831 DEBUG     yangsuite.ysnetconf.rpcbuilder : 402]
  ...minus pfx/localname: [subscription-id="105"]/mdt-cfg:base/mdt-cfg:encoding
[2021-04-27 00:00:20.831 DEBUG     yangsuite.ysnetconf.rpcbuilder : 433]
  ...minus keys / values: /mdt-cfg:base/mdt-cfg:encoding
[2021-04-27 00:00:20.832 DEBUG     yangsuite.ysnetconf.rpcbuilder : 378]
Parsing remnant of xpath: /mdt-cfg:base/mdt-cfg:encoding
[2021-04-27 00:00:20.833 DEBUG     yangsuite.ysnetconf.rpcbuilder : 402]
  ...minus pfx/localname: /mdt-cfg:encoding
[2021-04-27 00:00:20.833 DEBUG     yangsuite.ysnetconf.rpcbuilder : 378]
Parsing remnant of xpath: /mdt-cfg:encoding
[2021-04-27 00:00:20.834 DEBUG     yangsuite.ysnetconf.rpcbuilder : 402]
  ...minus pfx/localname: 
[2021-04-27 00:00:20.834 DEBUG     yangsuite.ysnetconf.rpcbuilder : 378]
Parsing remnant of xpath: /mdt-cfg:mdt-config-data/mdt-cfg:mdt-subscription[subscription-id="105"]/mdt-cfg:base/mdt-cfg:source-address
[2021-04-27 00:00:20.835 DEBUG     yangsuite.ysnetconf.rpcbuilder : 402]
  ...minus pfx/localname: /mdt-cfg:mdt-subscription[subscription-id="105"]/mdt-cfg:base/mdt-cfg:source-address
[2021-04-27 00:00:20.835 DEBUG     yangsuite.ysnetconf.rpcbuilder : 378]
Parsing remnant of xpath: /mdt-cfg:mdt-subscription[subscription-id="105"]/mdt-cfg:base/mdt-cfg:source-address
[2021-04-27 00:00:20.835 DEBUG     yangsuite.ysnetconf.rpcbuilder : 402]
  ...minus pfx/localname: [subscription-id="105"]/mdt-cfg:base/mdt-cfg:source-address
[2021-04-27 00:00:20.836 DEBUG     yangsuite.ysnetconf.rpcbuilder : 433]
  ...minus keys / values: /mdt-cfg:base/mdt-cfg:source-address
[2021-04-27 00:00:20.836 DEBUG     yangsuite.ysnetconf.rpcbuilder : 378]
Parsing remnant of xpath: /mdt-cfg:base/mdt-cfg:source-address
[2021-04-27 00:00:20.837 DEBUG     yangsuite.ysnetconf.rpcbuilder : 402]
  ...minus pfx/localname: /mdt-cfg:source-address
[2021-04-27 00:00:20.837 DEBUG     yangsuite.ysnetconf.rpcbuilder : 378]
Parsing remnant of xpath: /mdt-cfg:source-address
[2021-04-27 00:00:20.838 DEBUG     yangsuite.ysnetconf.rpcbuilder : 402]
  ...minus pfx/localname: 
[2021-04-27 00:00:20.839 DEBUG     yangsuite.ysnetconf.rpcbuilder : 378]
Parsing remnant of xpath: /mdt-cfg:mdt-config-data/mdt-cfg:mdt-subscription[subscription-id="105"]/mdt-cfg:base/mdt-cfg:source-address
[2021-04-27 00:00:20.839 DEBUG     yangsuite.ysnetconf.rpcbuilder : 402]
  ...minus pfx/localname: /mdt-cfg:mdt-subscription[subscription-id="105"]/mdt-cfg:base/mdt-cfg:source-address
[2021-04-27 00:00:20.840 DEBUG     yangsuite.ysnetconf.rpcbuilder : 378]
Parsing remnant of xpath: /mdt-cfg:mdt-subscription[subscription-id="105"]/mdt-cfg:base/mdt-cfg:source-address
[2021-04-27 00:00:20.841 DEBUG     yangsuite.ysnetconf.rpcbuilder : 402]
  ...minus pfx/localname: [subscription-id="105"]/mdt-cfg:base/mdt-cfg:source-address
[2021-04-27 00:00:20.842 DEBUG     yangsuite.ysnetconf.rpcbuilder : 433]
  ...minus keys / values: /mdt-cfg:base/mdt-cfg:source-address
[2021-04-27 00:00:20.844 DEBUG     yangsuite.ysnetconf.rpcbuilder : 378]
Parsing remnant of xpath: /mdt-cfg:base/mdt-cfg:source-address
[2021-04-27 00:00:20.846 DEBUG     yangsuite.ysnetconf.rpcbuilder : 402]
  ...minus pfx/localname: /mdt-cfg:source-address
[2021-04-27 00:00:20.848 DEBUG     yangsuite.ysnetconf.rpcbuilder : 378]
Parsing remnant of xpath: /mdt-cfg:source-address
[2021-04-27 00:00:20.849 DEBUG     yangsuite.ysnetconf.rpcbuilder : 402]
  ...minus pfx/localname: 
[2021-04-27 00:00:20.850 DEBUG     yangsuite.ysnetconf.rpcbuilder : 378]
Parsing remnant of xpath: /mdt-cfg:mdt-config-data/mdt-cfg:mdt-subscription[subscription-id="105"]/mdt-cfg:base/mdt-cfg:no-synch-on-start
[2021-04-27 00:00:20.851 DEBUG     yangsuite.ysnetconf.rpcbuilder : 402]
  ...minus pfx/localname: /mdt-cfg:mdt-subscription[subscription-id="105"]/mdt-cfg:base/mdt-cfg:no-synch-on-start
[2021-04-27 00:00:20.853 DEBUG     yangsuite.ysnetconf.rpcbuilder : 378]
Parsing remnant of xpath: /mdt-cfg:mdt-subscription[subscription-id="105"]/mdt-cfg:base/mdt-cfg:no-synch-on-start
[2021-04-27 00:00:20.853 DEBUG     yangsuite.ysnetconf.rpcbuilder : 402]
  ...minus pfx/localname: [subscription-id="105"]/mdt-cfg:base/mdt-cfg:no-synch-on-start
[2021-04-27 00:00:20.854 DEBUG     yangsuite.ysnetconf.rpcbuilder : 433]
  ...minus keys / values: /mdt-cfg:base/mdt-cfg:no-synch-on-start
[2021-04-27 00:00:20.854 DEBUG     yangsuite.ysnetconf.rpcbuilder : 378]
Parsing remnant of xpath: /mdt-cfg:base/mdt-cfg:no-synch-on-start
[2021-04-27 00:00:20.855 DEBUG     yangsuite.ysnetconf.rpcbuilder : 402]
  ...minus pfx/localname: /mdt-cfg:no-synch-on-start
[2021-04-27 00:00:20.855 DEBUG     yangsuite.ysnetconf.rpcbuilder : 378]
Parsing remnant of xpath: /mdt-cfg:no-synch-on-start
[2021-04-27 00:00:20.856 DEBUG     yangsuite.ysnetconf.rpcbuilder : 402]
  ...minus pfx/localname: 
[2021-04-27 00:00:20.857 DEBUG     yangsuite.ysnetconf.rpcbuilder : 378]
Parsing remnant of xpath: /mdt-cfg:mdt-config-data/mdt-cfg:mdt-subscription[subscription-id="105"]/mdt-cfg:base/mdt-cfg:no-synch-on-start
[2021-04-27 00:00:20.857 DEBUG     yangsuite.ysnetconf.rpcbuilder : 402]
  ...minus pfx/localname: /mdt-cfg:mdt-subscription[subscription-id="105"]/mdt-cfg:base/mdt-cfg:no-synch-on-start
[2021-04-27 00:00:20.858 DEBUG     yangsuite.ysnetconf.rpcbuilder : 378]
Parsing remnant of xpath: /mdt-cfg:mdt-subscription[subscription-id="105"]/mdt-cfg:base/mdt-cfg:no-synch-on-start
[2021-04-27 00:00:20.859 DEBUG     yangsuite.ysnetconf.rpcbuilder : 402]
  ...minus pfx/localname: [subscription-id="105"]/mdt-cfg:base/mdt-cfg:no-synch-on-start
[2021-04-27 00:00:20.859 DEBUG     yangsuite.ysnetconf.rpcbuilder : 433]
  ...minus keys / values: /mdt-cfg:base/mdt-cfg:no-synch-on-start
[2021-04-27 00:00:20.860 DEBUG     yangsuite.ysnetconf.rpcbuilder : 378]
Parsing remnant of xpath: /mdt-cfg:base/mdt-cfg:no-synch-on-start
[2021-04-27 00:00:20.860 DEBUG     yangsuite.ysnetconf.rpcbuilder : 402]
  ...minus pfx/localname: /mdt-cfg:no-synch-on-start
[2021-04-27 00:00:20.861 DEBUG     yangsuite.ysnetconf.rpcbuilder : 378]
Parsing remnant of xpath: /mdt-cfg:no-synch-on-start
[2021-04-27 00:00:20.861 DEBUG     yangsuite.ysnetconf.rpcbuilder : 402]
  ...minus pfx/localname: 
[2021-04-27 00:00:20.862 DEBUG     yangsuite.ysnetconf.rpcbuilder : 378]
Parsing remnant of xpath: /mdt-cfg:mdt-config-data/mdt-cfg:mdt-subscription[subscription-id="105"]/mdt-cfg:base/mdt-cfg:xpath
[2021-04-27 00:00:20.862 DEBUG     yangsuite.ysnetconf.rpcbuilder : 402]
  ...minus pfx/localname: /mdt-cfg:mdt-subscription[subscription-id="105"]/mdt-cfg:base/mdt-cfg:xpath
[2021-04-27 00:00:20.864 DEBUG     yangsuite.ysnetconf.rpcbuilder : 378]
Parsing remnant of xpath: /mdt-cfg:mdt-subscription[subscription-id="105"]/mdt-cfg:base/mdt-cfg:xpath
[2021-04-27 00:00:20.865 DEBUG     yangsuite.ysnetconf.rpcbuilder : 402]
  ...minus pfx/localname: [subscription-id="105"]/mdt-cfg:base/mdt-cfg:xpath
[2021-04-27 00:00:20.866 DEBUG     yangsuite.ysnetconf.rpcbuilder : 433]
  ...minus keys / values: /mdt-cfg:base/mdt-cfg:xpath
[2021-04-27 00:00:20.866 DEBUG     yangsuite.ysnetconf.rpcbuilder : 378]
Parsing remnant of xpath: /mdt-cfg:base/mdt-cfg:xpath
[2021-04-27 00:00:20.867 DEBUG     yangsuite.ysnetconf.rpcbuilder : 402]
  ...minus pfx/localname: /mdt-cfg:xpath
[2021-04-27 00:00:20.868 DEBUG     yangsuite.ysnetconf.rpcbuilder : 378]
Parsing remnant of xpath: /mdt-cfg:xpath
[2021-04-27 00:00:20.868 DEBUG     yangsuite.ysnetconf.rpcbuilder : 402]
  ...minus pfx/localname: 
[2021-04-27 00:00:20.869 DEBUG     yangsuite.ysnetconf.rpcbuilder : 378]
Parsing remnant of xpath: /mdt-cfg:mdt-config-data/mdt-cfg:mdt-subscription[subscription-id="105"]/mdt-cfg:base/mdt-cfg:xpath
[2021-04-27 00:00:20.870 DEBUG     yangsuite.ysnetconf.rpcbuilder : 402]
  ...minus pfx/localname: /mdt-cfg:mdt-subscription[subscription-id="105"]/mdt-cfg:base/mdt-cfg:xpath
[2021-04-27 00:00:20.870 DEBUG     yangsuite.ysnetconf.rpcbuilder : 378]
Parsing remnant of xpath: /mdt-cfg:mdt-subscription[subscription-id="105"]/mdt-cfg:base/mdt-cfg:xpath
[2021-04-27 00:00:20.873 DEBUG     yangsuite.ysnetconf.rpcbuilder : 402]
  ...minus pfx/localname: [subscription-id="105"]/mdt-cfg:base/mdt-cfg:xpath
[2021-04-27 00:00:20.874 DEBUG     yangsuite.ysnetconf.rpcbuilder : 433]
  ...minus keys / values: /mdt-cfg:base/mdt-cfg:xpath
[2021-04-27 00:00:20.875 DEBUG     yangsuite.ysnetconf.rpcbuilder : 378]
Parsing remnant of xpath: /mdt-cfg:base/mdt-cfg:xpath
[2021-04-27 00:00:20.875 DEBUG     yangsuite.ysnetconf.rpcbuilder : 402]
  ...minus pfx/localname: /mdt-cfg:xpath
[2021-04-27 00:00:20.876 DEBUG     yangsuite.ysnetconf.rpcbuilder : 378]
Parsing remnant of xpath: /mdt-cfg:xpath
[2021-04-27 00:00:20.878 DEBUG     yangsuite.ysnetconf.rpcbuilder : 402]
  ...minus pfx/localname: 
[2021-04-27 00:00:20.879 DEBUG     yangsuite.ysnetconf.rpcbuilder : 378]
Parsing remnant of xpath: /mdt-cfg:mdt-config-data/mdt-cfg:mdt-subscription[subscription-id="105"]/mdt-cfg:mdt-receivers[address="192.168.42.128"][port="57000"]/mdt-cfg:protocol
[2021-04-27 00:00:20.879 DEBUG     yangsuite.ysnetconf.rpcbuilder : 402]
  ...minus pfx/localname: /mdt-cfg:mdt-subscription[subscription-id="105"]/mdt-cfg:mdt-receivers[address="192.168.42.128"][port="57000"]/mdt-cfg:protocol
[2021-04-27 00:00:20.880 DEBUG     yangsuite.ysnetconf.rpcbuilder : 378]
Parsing remnant of xpath: /mdt-cfg:mdt-subscription[subscription-id="105"]/mdt-cfg:mdt-receivers[address="192.168.42.128"][port="57000"]/mdt-cfg:protocol
[2021-04-27 00:00:20.882 DEBUG     yangsuite.ysnetconf.rpcbuilder : 402]
  ...minus pfx/localname: [subscription-id="105"]/mdt-cfg:mdt-receivers[address="192.168.42.128"][port="57000"]/mdt-cfg:protocol
[2021-04-27 00:00:20.883 DEBUG     yangsuite.ysnetconf.rpcbuilder : 433]
  ...minus keys / values: /mdt-cfg:mdt-receivers[address="192.168.42.128"][port="57000"]/mdt-cfg:protocol
[2021-04-27 00:00:20.883 DEBUG     yangsuite.ysnetconf.rpcbuilder : 378]
Parsing remnant of xpath: /mdt-cfg:mdt-receivers[address="192.168.42.128"][port="57000"]/mdt-cfg:protocol
[2021-04-27 00:00:20.884 DEBUG     yangsuite.ysnetconf.rpcbuilder : 402]
  ...minus pfx/localname: [address="192.168.42.128"][port="57000"]/mdt-cfg:protocol
[2021-04-27 00:00:20.885 DEBUG     yangsuite.ysnetconf.rpcbuilder : 433]
  ...minus keys / values: [port="57000"]/mdt-cfg:protocol
[2021-04-27 00:00:20.885 DEBUG     yangsuite.ysnetconf.rpcbuilder : 433]
  ...minus keys / values: /mdt-cfg:protocol
[2021-04-27 00:00:20.886 DEBUG     yangsuite.ysnetconf.rpcbuilder : 378]
Parsing remnant of xpath: /mdt-cfg:protocol
[2021-04-27 00:00:20.886 DEBUG     yangsuite.ysnetconf.rpcbuilder : 402]
  ...minus pfx/localname: 
[2021-04-27 00:00:20.887 DEBUG     yangsuite.ysnetconf.rpcbuilder : 378]
Parsing remnant of xpath: /mdt-cfg:mdt-config-data/mdt-cfg:mdt-subscription[subscription-id="105"]/mdt-cfg:mdt-receivers[address="192.168.42.128"][port="57000"]/mdt-cfg:protocol
[2021-04-27 00:00:20.888 DEBUG     yangsuite.ysnetconf.rpcbuilder : 402]
  ...minus pfx/localname: /mdt-cfg:mdt-subscription[subscription-id="105"]/mdt-cfg:mdt-receivers[address="192.168.42.128"][port="57000"]/mdt-cfg:protocol
[2021-04-27 00:00:20.889 DEBUG     yangsuite.ysnetconf.rpcbuilder : 378]
Parsing remnant of xpath: /mdt-cfg:mdt-subscription[subscription-id="105"]/mdt-cfg:mdt-receivers[address="192.168.42.128"][port="57000"]/mdt-cfg:protocol
[2021-04-27 00:00:20.890 DEBUG     yangsuite.ysnetconf.rpcbuilder : 402]
  ...minus pfx/localname: [subscription-id="105"]/mdt-cfg:mdt-receivers[address="192.168.42.128"][port="57000"]/mdt-cfg:protocol
[2021-04-27 00:00:20.891 DEBUG     yangsuite.ysnetconf.rpcbuilder : 433]
  ...minus keys / values: /mdt-cfg:mdt-receivers[address="192.168.42.128"][port="57000"]/mdt-cfg:protocol
[2021-04-27 00:00:20.892 DEBUG     yangsuite.ysnetconf.rpcbuilder : 378]
Parsing remnant of xpath: /mdt-cfg:mdt-receivers[address="192.168.42.128"][port="57000"]/mdt-cfg:protocol
[2021-04-27 00:00:20.893 DEBUG     yangsuite.ysnetconf.rpcbuilder : 402]
  ...minus pfx/localname: [address="192.168.42.128"][port="57000"]/mdt-cfg:protocol
[2021-04-27 00:00:20.894 DEBUG     yangsuite.ysnetconf.rpcbuilder : 433]
  ...minus keys / values: [port="57000"]/mdt-cfg:protocol
[2021-04-27 00:00:20.894 DEBUG     yangsuite.ysnetconf.rpcbuilder : 433]
  ...minus keys / values: /mdt-cfg:protocol
[2021-04-27 00:00:20.895 DEBUG     yangsuite.ysnetconf.rpcbuilder : 378]
Parsing remnant of xpath: /mdt-cfg:protocol
[2021-04-27 00:00:20.895 DEBUG     yangsuite.ysnetconf.rpcbuilder : 402]
  ...minus pfx/localname: 
[2021-04-27 00:00:20.897 DEBUG     yangsuite.ysnetconf.rpcbuilder : 262]
get_payload: constructed XML:
<config xmlns="urn:ietf:params:xml:ns:netconf:base:1.0">
  <mdt-config-data xmlns="http://cisco.com/ns/yang/Cisco-IOS-XE-mdt-cfg">
    <mdt-subscription>
      <subscription-id>105</subscription-id>
      <base>
        <stream/>
        <encoding>encode-kvgpb</encoding>
        <source-address>192.168.42.82</source-address>
        <no-synch-on-start>true</no-synch-on-start>
        <xpath>/if:interfaces/interface/enabled</xpath>
      </base>
      <mdt-receivers>
        <address>192.168.42.128</address>
        <port>57000</port>
        <protocol>grpc-tcp</protocol>
      </mdt-receivers>
    </mdt-subscription>
  </mdt-config-data>
</config>

[2021-04-27 00:00:20.899 INFO                       django.server : 124]
"POST /netconf/getrpc/ HTTP/1.1" 200 884
[2021-04-27 04:13:45.674 INFO                       django.server : 124]
"GET / HTTP/1.1" 302 0
[2021-04-27 04:13:45.715 WARNING     yangsuite.context_processors :  78]
Core app "ysfilemanager" not found in application registry!
[2021-04-27 04:13:45.718 WARNING     yangsuite.context_processors :  78]
Core app "ysdevices" not found in application registry!
[2021-04-27 04:13:45.719 WARNING     yangsuite.context_processors :  78]
Core app "ysyangtree" not found in application registry!
[2021-04-27 04:13:45.719 DEBUG       yangsuite.context_processors : 136]
Adding menus from yangsuite
[2021-04-27 04:13:45.720 DEBUG       yangsuite.context_processors : 172]
Added menu 'Admin'; menus are now ('Admin',)
[2021-04-27 04:13:45.720 DEBUG       yangsuite.context_processors : 186]
Added menu item 'Manage users → /admin' in menu 'Admin'
[2021-04-27 04:13:45.720 DEBUG       yangsuite.context_processors : 186]
Added menu item 'Manage plugins → /yangsuite/plugins/' in menu 'Admin'
[2021-04-27 04:13:45.721 DEBUG       yangsuite.context_processors : 186]
Added menu item 'View logs → /yangsuite/logs' in menu 'Admin'
[2021-04-27 04:13:45.721 DEBUG       yangsuite.context_processors : 172]
Added menu 'Setup'; menus are now ('Admin', 'Setup')
[2021-04-27 04:13:45.722 INFO        yangsuite.context_processors : 121]
yangsuite_menus is now: OrderedDict([('Admin', [YSMenuItem(text='Manage users', url='/admin'), YSMenuItem(text='Manage plugins', url='/yangsuite/plugins/'), YSMenuItem(text='View logs', url='/yangsuite/logs')]), ('Setup', [])])
[2021-04-27 04:13:45.731 INFO                       django.server : 124]
"GET /yangsuite/plugins/ HTTP/1.1" 200 8463
[2021-04-27 04:13:45.803 INFO                       django.server : 124]
"GET /static/yangsuite/js/jquery-1.11.2.min.js HTTP/1.1" 304 0
[2021-04-27 04:13:45.823 INFO                       django.server : 124]
"GET /static/yangsuite/js/jstree/themes/default/style.min.css HTTP/1.1" 304 0
[2021-04-27 04:13:45.824 INFO                       django.server : 124]
"GET /static/yangsuite/css/cui-standard-1.3.3.min.css HTTP/1.1" 304 0
[2021-04-27 04:13:45.824 INFO                       django.server : 124]
"GET /static/yangsuite/css/excite-bike/jquery-ui.min.css HTTP/1.1" 304 0
[2021-04-27 04:13:45.827 INFO                       django.server : 124]
"GET /static/yangsuite/css/yangsuite-cui.css HTTP/1.1" 304 0
[2021-04-27 04:13:45.837 INFO                       django.server : 124]
"GET /static/yangsuite/css/yangsuite_plugins.css HTTP/1.1" 304 0
[2021-04-27 04:13:45.843 INFO                       django.server : 124]
"GET /static/yangsuite/js/jquery-ui.min.js HTTP/1.1" 304 0
[2021-04-27 04:13:45.844 INFO                       django.server : 124]
"GET /static/yangsuite/js/plugins.js HTTP/1.1" 304 0
[2021-04-27 04:13:45.847 INFO                       django.server : 124]
"GET /static/yangsuite/js/jstree/jstree.js HTTP/1.1" 304 0
[2021-04-27 04:13:45.858 INFO                       django.server : 124]
"GET /static/yangsuite/js/js.cookie.js HTTP/1.1" 304 0
[2021-04-27 04:13:45.863 INFO                       django.server : 124]
"GET /static/yangsuite/js/jstree/jstreegrid.js HTTP/1.1" 304 0
[2021-04-27 04:13:45.866 INFO                       django.server : 124]
"GET /static/yangsuite/js/yangsuite.js HTTP/1.1" 304 0
[2021-04-27 04:13:45.907 INFO                       django.server : 124]
"GET /static/yangsuite/favicon/yangsuite.png HTTP/1.1" 304 0
[2021-04-27 04:13:45.923 INFO                       django.server : 124]
"POST /gtm HTTP/1.1" 200 20
[2021-04-27 04:13:46.124 INFO                       django.server : 124]
"GET /static/yangsuite/fonts/CiscoSans/CiscoSansTTLight.woff2 HTTP/1.1" 304 0
[2021-04-27 04:13:46.133 INFO                   yangsuite.plugins :  75]
Installed plugins: ['yangsuite', 'yangsuite-devices', 'yangsuite-filemanager', 'yangsuite-netconf', 'yangsuite-yangtree']
[2021-04-27 04:13:46.137 INFO                       django.server : 124]
"GET /yangsuite/plugins/mode HTTP/1.1" 200 15
[2021-04-27 04:13:46.142 INFO                       django.server : 124]
"GET /static/yangsuite/fonts/CiscoSans/CiscoSansTTRegular.woff2 HTTP/1.1" 304 0
[2021-04-27 04:13:46.149 INFO                       django.server : 124]
"GET /static/yangsuite/fonts/cui-font-1.3.3.woff2 HTTP/1.1" 304 0
[2021-04-27 04:13:46.151 INFO                       django.server : 124]
"GET /static/yangsuite/fonts/CiscoSans/CiscoSansTTBold.woff2 HTTP/1.1" 304 0
[2021-04-27 04:13:46.171 INFO                       django.server : 124]
"GET /yangsuite/plugins/list HTTP/1.1" 200 1301
[2021-04-27 04:13:46.180 INFO                       django.server : 124]
"GET /static/yangsuite/fonts/CiscoSans/CiscoSansTTExtraLight.woff2 HTTP/1.1" 304 0
[2021-04-27 04:13:46.233 INFO                       django.server : 124]
"GET /static/yangsuite/css/excite-bike/images/ui-icons_0a82eb_256x240.png HTTP/1.1" 304 0
[2021-04-27 04:13:46.233 INFO                       django.server : 124]
"GET /static/yangsuite/css/excite-bike/images/ui-icons_fcdd4a_256x240.png HTTP/1.1" 304 0
[2021-04-27 04:13:46.280 INFO                       django.server : 124]
"GET /static/yangsuite/css/excite-bike/images/ui-icons_ffffff_256x240.png HTTP/1.1" 304 0
[2021-04-27 04:13:46.740 INFO                       django.server : 124]
"GET /static/yangsuite/favicon/manifest.json HTTP/1.1" 304 0
[2021-04-27 04:13:47.903 INFO                       django.server : 124]
"POST /yangsuite/plugins/report HTTP/1.1" 200 2482
[2021-04-27 04:13:48.540 INFO                       django.server : 124]
"GET /yangsuite/plugins/check_versions HTTP/1.1" 200 516
[2021-04-27 04:13:57.791 INFO                       django.server : 124]
"GET /yangsuite/plugins/ HTTP/1.1" 200 8463
[2021-04-27 04:13:57.875 INFO                       django.server : 124]
"POST /gtm HTTP/1.1" 200 20
[2021-04-27 04:13:58.030 INFO                   yangsuite.plugins :  75]
Installed plugins: ['yangsuite', 'yangsuite-devices', 'yangsuite-filemanager', 'yangsuite-netconf', 'yangsuite-yangtree']
[2021-04-27 04:13:58.032 INFO                       django.server : 124]
"GET /yangsuite/plugins/list HTTP/1.1" 200 1301
[2021-04-27 04:13:58.035 INFO                       django.server : 124]
"GET /yangsuite/plugins/mode HTTP/1.1" 200 15
[2021-04-27 04:13:58.052 INFO                       django.server : 124]
"GET /static/yangsuite/favicon/favicon.ico HTTP/1.1" 200 15086
[2021-04-27 04:13:58.545 INFO                       django.server : 124]
"GET /yangsuite/plugins/check_versions HTTP/1.1" 200 516
[2021-04-27 04:13:59.301 INFO                       django.server : 124]
"POST /yangsuite/plugins/report HTTP/1.1" 200 2482
[2021-04-27 04:14:51.868 DEBUG                     yangsuite.urls :  90]
Including ysyangtree.urls at 'yangtree'
[2021-04-27 04:14:51.874 DEBUG                     yangsuite.urls :  90]
Including ysnetconf.urls at 'netconf'
[2021-04-27 04:14:52.001 DEBUG                     yangsuite.urls :  90]
Including ysfilemanager.urls at 'filemanager'
[2021-04-27 04:14:52.003 DEBUG                     yangsuite.urls :  90]
Including ysdevices.urls at 'devices'
[2021-04-27 04:14:53.993 DEBUG                     yangsuite.urls :  90]
Including ysyangtree.urls at 'yangtree'
[2021-04-27 04:14:53.998 DEBUG                     yangsuite.urls :  90]
Including ysnetconf.urls at 'netconf'
[2021-04-27 04:14:54.066 DEBUG                     yangsuite.urls :  90]
Including ysfilemanager.urls at 'filemanager'
[2021-04-27 04:14:54.067 DEBUG                     yangsuite.urls :  90]
Including ysdevices.urls at 'devices'
[2021-04-27 04:15:15.385 DEBUG                     yangsuite.urls :  90]
Including ysyangtree.urls at 'yangtree'
[2021-04-27 04:15:15.391 DEBUG                     yangsuite.urls :  90]
Including ysnetconf.urls at 'netconf'
[2021-04-27 04:15:15.467 DEBUG                     yangsuite.urls :  90]
Including ysfilemanager.urls at 'filemanager'
[2021-04-27 04:15:15.469 DEBUG                     yangsuite.urls :  90]
Including ysdevices.urls at 'devices'
[2021-04-27 04:15:18.439 DEBUG                     yangsuite.urls :  90]
Including ysyangtree.urls at 'yangtree'
[2021-04-27 04:15:18.445 DEBUG                     yangsuite.urls :  90]
Including ysnetconf.urls at 'netconf'
[2021-04-27 04:15:18.542 DEBUG                     yangsuite.urls :  90]
Including ysfilemanager.urls at 'filemanager'
[2021-04-27 04:15:18.545 DEBUG                     yangsuite.urls :  90]
Including ysdevices.urls at 'devices'
[2021-04-27 04:18:56.237 DEBUG                     yangsuite.urls :  90]
Including ysyangtree.urls at 'yangtree'
[2021-04-27 04:18:56.241 DEBUG                     yangsuite.urls :  90]
Including ysnetconf.urls at 'netconf'
[2021-04-27 04:18:56.333 DEBUG                     yangsuite.urls :  90]
Including ysfilemanager.urls at 'filemanager'
[2021-04-27 04:18:56.335 DEBUG                     yangsuite.urls :  90]
Including ysdevices.urls at 'devices'
[2021-04-27 04:18:58.750 DEBUG                     yangsuite.urls :  90]
Including ysyangtree.urls at 'yangtree'
[2021-04-27 04:18:58.754 DEBUG                     yangsuite.urls :  90]
Including ysnetconf.urls at 'netconf'
[2021-04-27 04:18:58.852 DEBUG                     yangsuite.urls :  90]
Including ysfilemanager.urls at 'filemanager'
[2021-04-27 04:18:58.853 DEBUG                     yangsuite.urls :  90]
Including ysdevices.urls at 'devices'
[2021-04-27 04:19:29.506 DEBUG                     yangsuite.urls :  90]
Including ysyangtree.urls at 'yangtree'
[2021-04-27 04:19:29.511 DEBUG                     yangsuite.urls :  90]
Including ysnetconf.urls at 'netconf'
[2021-04-27 04:19:29.590 DEBUG                     yangsuite.urls :  90]
Including ysfilemanager.urls at 'filemanager'
[2021-04-27 04:19:29.591 DEBUG                     yangsuite.urls :  90]
Including ysdevices.urls at 'devices'
[2021-04-27 04:19:32.173 DEBUG                     yangsuite.urls :  90]
Including ysyangtree.urls at 'yangtree'
[2021-04-27 04:19:32.178 DEBUG                     yangsuite.urls :  90]
Including ysnetconf.urls at 'netconf'
[2021-04-27 04:19:32.269 DEBUG                     yangsuite.urls :  90]
Including ysfilemanager.urls at 'filemanager'
[2021-04-27 04:19:32.271 DEBUG                     yangsuite.urls :  90]
Including ysdevices.urls at 'devices'
[2021-04-27 04:20:42.285 DEBUG                     yangsuite.urls :  90]
Including ysyangtree.urls at 'yangtree'
[2021-04-27 04:20:42.288 DEBUG                     yangsuite.urls :  90]
Including ysnetconf.urls at 'netconf'
[2021-04-27 04:20:42.350 DEBUG                     yangsuite.urls :  90]
Including ysfilemanager.urls at 'filemanager'
[2021-04-27 04:20:42.352 DEBUG                     yangsuite.urls :  90]
Including ysdevices.urls at 'devices'
[2021-04-27 04:20:44.241 DEBUG                     yangsuite.urls :  90]
Including ysyangtree.urls at 'yangtree'
[2021-04-27 04:20:44.247 DEBUG                     yangsuite.urls :  90]
Including ysnetconf.urls at 'netconf'
[2021-04-27 04:20:44.307 DEBUG                     yangsuite.urls :  90]
Including ysfilemanager.urls at 'filemanager'
[2021-04-27 04:20:44.309 DEBUG                     yangsuite.urls :  90]
Including ysdevices.urls at 'devices'
[2021-04-27 04:20:55.686 DEBUG       yangsuite.context_processors : 136]
Adding menus from yangsuite
[2021-04-27 04:20:55.690 DEBUG       yangsuite.context_processors : 172]
Added menu 'Admin'; menus are now ('Admin',)
[2021-04-27 04:20:55.695 DEBUG       yangsuite.context_processors : 186]
Added menu item 'Manage users → /admin' in menu 'Admin'
[2021-04-27 04:20:55.696 DEBUG       yangsuite.context_processors : 186]
Added menu item 'Manage plugins → /yangsuite/plugins/' in menu 'Admin'
[2021-04-27 04:20:55.696 DEBUG       yangsuite.context_processors : 186]
Added menu item 'View logs → /yangsuite/logs' in menu 'Admin'
[2021-04-27 04:20:55.696 DEBUG       yangsuite.context_processors : 172]
Added menu 'Setup'; menus are now ('Admin', 'Setup')
[2021-04-27 04:20:55.697 DEBUG       yangsuite.context_processors : 136]
Adding menus from ysfilemanager
[2021-04-27 04:20:55.697 DEBUG       yangsuite.context_processors : 186]
Added menu item 'YANG files and repositories → /filemanager/repository/manage/' in menu 'Setup'
[2021-04-27 04:20:55.697 DEBUG       yangsuite.context_processors : 186]
Added menu item 'YANG module sets → /filemanager/yangsets/manage/' in menu 'Setup'
[2021-04-27 04:20:55.697 DEBUG       yangsuite.context_processors : 136]
Adding menus from ysdevices
[2021-04-27 04:20:55.697 DEBUG       yangsuite.context_processors : 186]
Added menu item 'Device profiles → /devices/devprofile' in menu 'Setup'
[2021-04-27 04:20:55.697 DEBUG       yangsuite.context_processors : 136]
Adding menus from ysyangtree
[2021-04-27 04:20:55.698 DEBUG       yangsuite.context_processors : 172]
Added menu 'Explore'; menus are now ('Admin', 'Setup', 'Explore')
[2021-04-27 04:20:55.698 DEBUG       yangsuite.context_processors : 186]
Added menu item 'YANG → /yangtree/explore' in menu 'Explore'
[2021-04-27 04:20:55.698 DEBUG       yangsuite.context_processors : 136]
Adding menus from ysnetconf
[2021-04-27 04:20:55.698 DEBUG       yangsuite.context_processors : 172]
Added menu 'Protocols'; menus are now ('Admin', 'Setup', 'Explore', 'Protocols')
[2021-04-27 04:20:55.700 DEBUG       yangsuite.context_processors : 186]
Added menu item 'NETCONF → /netconf/getyang' in menu 'Protocols'
[2021-04-27 04:20:55.700 INFO        yangsuite.context_processors : 121]
yangsuite_menus is now: OrderedDict([('Admin', [YSMenuItem(text='Manage users', url='/admin'), YSMenuItem(text='Manage plugins', url='/yangsuite/plugins/'), YSMenuItem(text='View logs', url='/yangsuite/logs')]), ('Setup', [YSMenuItem(text='YANG files and repositories', url='/filemanager/repository/manage/'), YSMenuItem(text='YANG module sets', url='/filemanager/yangsets/manage/'), YSMenuItem(text='Device profiles', url='/devices/devprofile')]), ('Explore', [YSMenuItem(text='YANG', url='/yangtree/explore')]), ('Protocols', [YSMenuItem(text='NETCONF', url='/netconf/getyang')])])
[2021-04-27 04:20:55.741 INFO                       django.server : 124]
"GET /yangsuite/plugins/ HTTP/1.1" 200 9839
[2021-04-27 04:20:55.793 INFO                       django.server : 124]
"GET /static/yangsuite/js/jquery-1.11.2.min.js HTTP/1.1" 304 0
[2021-04-27 04:20:55.813 INFO                       django.server : 124]
"GET /static/yangsuite/css/yangsuite-cui.css HTTP/1.1" 304 0
[2021-04-27 04:20:55.817 INFO                       django.server : 124]
"GET /static/yangsuite/css/yangsuite_plugins.css HTTP/1.1" 304 0
[2021-04-27 04:20:55.821 INFO                       django.server : 124]
"GET /static/yangsuite/css/cui-standard-1.3.3.min.css HTTP/1.1" 304 0
[2021-04-27 04:20:55.821 INFO                       django.server : 124]
"GET /static/yangsuite/css/excite-bike/jquery-ui.min.css HTTP/1.1" 304 0
[2021-04-27 04:20:55.822 INFO                       django.server : 124]
"GET /static/yangsuite/js/js.cookie.js HTTP/1.1" 304 0
[2021-04-27 04:20:55.826 INFO                       django.server : 124]
"GET /static/yangsuite/js/jquery-ui.min.js HTTP/1.1" 304 0
[2021-04-27 04:20:55.839 INFO                       django.server : 124]
"GET /static/yangsuite/js/plugins.js HTTP/1.1" 304 0
[2021-04-27 04:20:55.844 INFO                       django.server : 124]
"GET /static/yangsuite/js/yangsuite.js HTTP/1.1" 304 0
[2021-04-27 04:20:55.848 INFO                       django.server : 124]
"GET /static/yangsuite/js/jstree/themes/default/style.min.css HTTP/1.1" 304 0
[2021-04-27 04:20:55.853 INFO                       django.server : 124]
"GET /static/yangsuite/js/jstree/jstreegrid.js HTTP/1.1" 304 0
[2021-04-27 04:20:55.854 INFO                       django.server : 124]
"GET /static/yangsuite/js/jstree/jstree.js HTTP/1.1" 304 0
[2021-04-27 04:20:55.933 INFO                       django.server : 124]
"GET /static/yangsuite/favicon/yangsuite.png HTTP/1.1" 304 0
[2021-04-27 04:20:55.974 INFO                       django.server : 124]
"POST /gtm HTTP/1.1" 200 20
[2021-04-27 04:20:56.223 INFO                       django.server : 124]
"GET /static/yangsuite/fonts/CiscoSans/CiscoSansTTLight.woff2 HTTP/1.1" 304 0
[2021-04-27 04:20:56.269 INFO                       django.server : 124]
"GET /static/yangsuite/fonts/CiscoSans/CiscoSansTTRegular.woff2 HTTP/1.1" 304 0
[2021-04-27 04:20:56.295 INFO                       django.server : 124]
"GET /static/yangsuite/fonts/CiscoSans/CiscoSansTTBold.woff2 HTTP/1.1" 304 0
[2021-04-27 04:20:56.333 INFO                       django.server : 124]
"GET /yangsuite/plugins/mode HTTP/1.1" 200 15
[2021-04-27 04:20:56.352 INFO                       django.server : 124]
"GET /static/yangsuite/fonts/CiscoSans/CiscoSansTTExtraLight.woff2 HTTP/1.1" 304 0
[2021-04-27 04:20:56.423 INFO                   yangsuite.plugins :  75]
Installed plugins: ['yangsuite', 'yangsuite-devices', 'yangsuite-filemanager', 'yangsuite-netconf', 'yangsuite-yangtree']
[2021-04-27 04:20:56.425 INFO                       django.server : 124]
"GET /yangsuite/plugins/list HTTP/1.1" 200 2211
[2021-04-27 04:20:56.448 INFO                       django.server : 124]
"GET /static/yangsuite/fonts/cui-font-1.3.3.woff2 HTTP/1.1" 304 0
[2021-04-27 04:20:56.554 INFO                       django.server : 124]
"GET /static/yangsuite/favicon/favicon.ico HTTP/1.1" 200 15086
[2021-04-27 04:20:56.563 INFO                       django.server : 124]
"GET /static/yangsuite/favicon/manifest.json HTTP/1.1" 304 0
[2021-04-27 04:20:56.913 INFO                       django.server : 124]
"GET /yangsuite/plugins/check_versions HTTP/1.1" 200 516
[2021-04-27 04:20:58.357 INFO                       django.server : 124]
"POST /yangsuite/plugins/report HTTP/1.1" 200 2504
[2021-04-27 04:21:09.419 INFO                       django.server : 124]
"GET /yangtree/explore/ HTTP/1.1" 200 19815
[2021-04-27 04:21:09.462 INFO                       django.server : 124]
"GET /static/ysfilemanager/js/yangsets.js HTTP/1.1" 304 0
[2021-04-27 04:21:09.462 INFO                       django.server : 124]
"GET /static/ysyangtree/js/yangtree.js HTTP/1.1" 304 0
[2021-04-27 04:21:09.475 INFO                       django.server : 124]
"GET /static/ysyangtree/js/tasks.js HTTP/1.1" 304 0
[2021-04-27 04:21:09.481 INFO                       django.server : 124]
"GET /static/ysyangtree/js/chosen/chosen.jquery.js HTTP/1.1" 304 0
[2021-04-27 04:21:09.495 INFO                       django.server : 124]
"GET /static/ysyangtree/css/yangtree.css HTTP/1.1" 304 0
[2021-04-27 04:21:09.499 INFO                       django.server : 124]
"GET /static/ysyangtree/css/chosen/chosen.css HTTP/1.1" 304 0
[2021-04-27 04:21:09.544 INFO                       django.server : 124]
"POST /gtm HTTP/1.1" 200 20
[2021-04-27 04:21:09.565 INFO                       django.server : 124]
"GET /static/yangsuite/js/fontawesome-all.js HTTP/1.1" 304 0
[2021-04-27 04:21:09.689 INFO                       django.server : 124]
"GET /yangsuite/plugins/mode HTTP/1.1" 200 15
[2021-04-27 04:21:09.734 INFO                       django.server : 124]
"GET /filemanager/yangsets/list/?readonly=true HTTP/1.1" 200 77
[2021-04-27 04:21:14.570 DEBUG   yangsuite.ysfilemanager.filemanager.utility :  76]
Split userset "nsindiso+csr100v" to user "nsindiso", setname/reponame "csr100v"
[2021-04-27 04:21:14.571 DEBUG   yangsuite.ysfilemanager.filemanager.utility :  76]
Split userset "nsindiso+csr100v" to user "nsindiso", setname/reponame "csr100v"
[2021-04-27 04:21:14.604 INFO    yangsuite.ysfilemanager.filemanager.yangset : 280]
Loaded YANG set 'CSR100v' owned by 'nsindiso' (543 modules) from /home/nsindiso/Documents/Model_Driven_Telemetry_Lab/[]/users/nsindiso/yangsets/csr100v
[2021-04-27 04:21:14.606 INFO                       django.server : 124]
"GET /filemanager/yangsets/get/?yangset=nsindiso%2Bcsr100v HTTP/1.1" 200 33088
[2021-04-27 04:21:31.752 INFO                       django.server : 124]
"GET /static/ysyangtree/css/chosen/chosen-sprite.png HTTP/1.1" 304 0
[2021-04-27 04:21:34.105 DEBUG   yangsuite.ysfilemanager.filemanager.utility :  76]
Split userset "nsindiso+csr100v" to user "nsindiso", setname/reponame "csr100v"
[2021-04-27 04:21:34.107 DEBUG   yangsuite.ysfilemanager.filemanager.utility :  76]
Split userset "nsindiso+csr100v" to user "nsindiso", setname/reponame "csr100v"
[2021-04-27 04:21:34.146 INFO    yangsuite.ysfilemanager.filemanager.yangset : 280]
Loaded YANG set 'CSR100v' owned by 'nsindiso' (543 modules) from /home/nsindiso/Documents/Model_Driven_Telemetry_Lab/[]/users/nsindiso/yangsets/csr100v
[2021-04-27 04:21:34.152 DEBUG   yangsuite.ysfilemanager.filemanager.utility :  76]
Split userset "nsindiso+csr100v" to user "nsindiso", setname/reponame "csr100v"
[2021-04-27 04:21:34.157 DEBUG   yangsuite.ysfilemanager.filemanager.utility :  76]
Split userset "nsindiso+csr100v" to user "nsindiso", setname/reponame "csr100v"
[2021-04-27 04:21:34.192 INFO    yangsuite.ysfilemanager.filemanager.yangset : 280]
Loaded YANG set 'CSR100v' owned by 'nsindiso' (543 modules) from /home/nsindiso/Documents/Model_Driven_Telemetry_Lab/[]/users/nsindiso/yangsets/csr100v
[2021-04-27 04:21:34.193 INFO        yangsuite.ysyangtree.context : 288]
Initializing context for path 'YANG set 'CSR100v' owned by 'nsindiso' (543 modules)'
[2021-04-27 04:21:34.194 INFO        yangsuite.ysyangtree.context : 296]
Context stored with reference 'nsindiso'
[2021-04-27 04:21:34.195 DEBUG       yangsuite.ysyangtree.context : 396]
No module loading requested.
[2021-04-27 04:21:34.196 INFO        yangsuite.ysyangtree.context : 340]
Context 'nsindiso+csr100v' now fully loaded and ready
[2021-04-27 04:21:34.961 DEBUG       yangsuite.ysyangtree.context : 400]
Loading module(s) {'openconfig-yang-types'} from repository YANG set 'CSR100v' owned by 'nsindiso' (543 modules)
[2021-04-27 04:21:34.962 DEBUG       yangsuite.ysyangtree.context : 407]
Will load upstream modules as well: ['cisco-xe-openconfig-acl-deviation', 'cisco-xe-openconfig-aft-deviation', 'cisco-xe-openconfig-bgp-deviation', 'cisco-xe-openconfig-bgp-policy-deviation', 'cisco-xe-openconfig-if-ethernet-ext', 'cisco-xe-openconfig-if-ip-deviation', 'cisco-xe-openconfig-interfaces-deviation', 'cisco-xe-openconfig-interfaces-ext', 'cisco-xe-openconfig-isis-deviation', 'cisco-xe-openconfig-lldp-deviation', 'cisco-xe-openconfig-mpls-deviation', 'cisco-xe-openconfig-network-instance-deviation', 'cisco-xe-openconfig-rib-bgp-ext', 'cisco-xe-openconfig-routing-policy-deviation', 'cisco-xe-openconfig-segment-routing-deviation', 'cisco-xe-openconfig-system-ext', 'cisco-xe-routing-csr-openconfig-platform-deviation', 'cisco-xe-routing-openconfig-system-deviation', 'cisco-xe-routing-openconfig-system-management-deviation', 'cisco-xe-routing-openconfig-vlan-deviation', 'openconfig-aaa', 'openconfig-aaa-radius', 'openconfig-acl', 'openconfig-aft', 'openconfig-alarms', 'openconfig-bgp', 'openconfig-bgp-common', 'openconfig-bgp-common-multiprotocol', 'openconfig-bgp-common-structure', 'openconfig-bgp-global', 'openconfig-bgp-neighbor', 'openconfig-bgp-peer-group', 'openconfig-bgp-policy', 'openconfig-if-aggregate', 'openconfig-if-ethernet', 'openconfig-if-ip', 'openconfig-if-ip-ext', 'openconfig-interfaces', 'openconfig-isis', 'openconfig-isis-policy', 'openconfig-isis-routing', 'openconfig-lacp', 'openconfig-lldp', 'openconfig-local-routing', 'openconfig-mpls', 'openconfig-mpls-rsvp', 'openconfig-mpls-te', 'openconfig-network-instance', 'openconfig-network-instance-l2', 'openconfig-packet-match', 'openconfig-platform', 'openconfig-routing-policy', 'openconfig-segment-routing', 'openconfig-system', 'openconfig-vlan']
[2021-04-27 04:21:34.962 WARNING     yangsuite.ysyangtree.context : 414]
Loading 56 YANG modules. This may take some time and consume significant amounts of system memory.
[2021-04-27 04:21:35.138 DEBUG   yangsuite.ysfilemanager.filemanager.utility :  76]
Split userset "nsindiso+csr100v" to user "nsindiso", setname/reponame "csr100v"
[2021-04-27 04:21:35.152 DEBUG   yangsuite.ysfilemanager.filemanager.utility :  76]
Split userset "nsindiso+csr100v" to user "nsindiso", setname/reponame "csr100v"
[2021-04-27 04:21:35.202 DEBUG       yangsuite.ysyangtree.context : 500]
Successfully loaded cisco-xe-openconfig-if-ethernet-ext
[2021-04-27 04:21:35.208 DEBUG       yangsuite.ysyangtree.context : 500]
Successfully loaded cisco-xe-openconfig-interfaces-ext
[2021-04-27 04:21:36.918 INFO    yangsuite.ysfilemanager.filemanager.yangset : 280]
Loaded YANG set 'CSR100v' owned by 'nsindiso' (543 modules) from /home/nsindiso/Documents/Model_Driven_Telemetry_Lab/[]/users/nsindiso/yangsets/csr100v
[2021-04-27 04:21:36.961 DEBUG       yangsuite.ysyangtree.context : 159]
Cached ctx has same yangset
[2021-04-27 04:21:36.974 INFO                       django.server : 124]
"GET /yangtree/gettree/?yangset=nsindiso%2Bcsr100v HTTP/1.1" 200 77
[2021-04-27 04:21:37.056 DEBUG   yangsuite.ysfilemanager.filemanager.utility :  76]
Split userset "nsindiso+csr100v" to user "nsindiso", setname/reponame "csr100v"
[2021-04-27 04:21:37.065 DEBUG   yangsuite.ysfilemanager.filemanager.utility :  76]
Split userset "nsindiso+csr100v" to user "nsindiso", setname/reponame "csr100v"
[2021-04-27 04:21:37.889 DEBUG       yangsuite.ysyangtree.context : 500]
Successfully loaded cisco-xe-openconfig-rib-bgp-ext
[2021-04-27 04:21:38.317 DEBUG       yangsuite.ysyangtree.context : 500]
Successfully loaded cisco-xe-openconfig-system-ext
[2021-04-27 04:21:38.335 DEBUG       yangsuite.ysyangtree.context : 500]
Successfully loaded openconfig-aaa
[2021-04-27 04:21:38.341 DEBUG       yangsuite.ysyangtree.context : 500]
Successfully loaded openconfig-aaa-radius
[2021-04-27 04:21:38.428 DEBUG       yangsuite.ysyangtree.context : 500]
Successfully loaded openconfig-acl
[2021-04-27 04:21:38.438 DEBUG       yangsuite.ysyangtree.context : 500]
Successfully loaded openconfig-aft
[2021-04-27 04:21:38.445 DEBUG       yangsuite.ysyangtree.context : 500]
Successfully loaded openconfig-alarms
[2021-04-27 04:21:38.449 DEBUG       yangsuite.ysyangtree.context : 500]
Successfully loaded openconfig-bgp
[2021-04-27 04:21:38.465 DEBUG       yangsuite.ysyangtree.context : 500]
Successfully loaded openconfig-bgp-common
[2021-04-27 04:21:38.476 DEBUG       yangsuite.ysyangtree.context : 500]
Successfully loaded openconfig-bgp-common-multiprotocol
[2021-04-27 04:21:38.482 DEBUG       yangsuite.ysyangtree.context : 500]
Successfully loaded openconfig-bgp-common-structure
[2021-04-27 04:21:38.490 DEBUG       yangsuite.ysyangtree.context : 500]
Successfully loaded openconfig-bgp-global
[2021-04-27 04:21:38.504 DEBUG       yangsuite.ysyangtree.context : 500]
Successfully loaded openconfig-bgp-neighbor
[2021-04-27 04:21:38.512 DEBUG       yangsuite.ysyangtree.context : 500]
Successfully loaded openconfig-bgp-peer-group
[2021-04-27 04:21:38.593 DEBUG       yangsuite.ysyangtree.context : 500]
Successfully loaded openconfig-bgp-policy
[2021-04-27 04:21:38.599 DEBUG       yangsuite.ysyangtree.context : 500]
Successfully loaded openconfig-if-aggregate
[2021-04-27 04:21:38.607 DEBUG       yangsuite.ysyangtree.context : 500]
Successfully loaded openconfig-if-ethernet
[2021-04-27 04:21:38.702 DEBUG       yangsuite.ysyangtree.context : 500]
Successfully loaded openconfig-if-ip
[2021-04-27 04:21:38.712 DEBUG       yangsuite.ysyangtree.context : 500]
Successfully loaded openconfig-if-ip-ext
[2021-04-27 04:21:38.735 DEBUG       yangsuite.ysyangtree.context : 500]
Successfully loaded openconfig-interfaces
[2021-04-27 04:21:38.778 DEBUG       yangsuite.ysyangtree.context : 500]
Successfully loaded openconfig-isis
[2021-04-27 04:21:38.787 DEBUG       yangsuite.ysyangtree.context : 500]
Successfully loaded openconfig-isis-policy
[2021-04-27 04:21:38.799 DEBUG       yangsuite.ysyangtree.context : 500]
Successfully loaded openconfig-isis-routing
[2021-04-27 04:21:38.826 DEBUG       yangsuite.ysyangtree.context : 500]
Successfully loaded openconfig-lacp
[2021-04-27 04:21:38.885 DEBUG       yangsuite.ysyangtree.context : 500]
Successfully loaded openconfig-lldp
[2021-04-27 04:21:38.894 DEBUG       yangsuite.ysyangtree.context : 500]
Successfully loaded openconfig-local-routing
[2021-04-27 04:21:38.909 DEBUG       yangsuite.ysyangtree.context : 500]
Successfully loaded openconfig-mpls
[2021-04-27 04:21:38.933 DEBUG       yangsuite.ysyangtree.context : 500]
Successfully loaded openconfig-mpls-rsvp
[2021-04-27 04:21:38.963 DEBUG       yangsuite.ysyangtree.context : 500]
Successfully loaded openconfig-mpls-te
[2021-04-27 04:21:38.993 DEBUG       yangsuite.ysyangtree.context : 500]
Successfully loaded openconfig-network-instance
[2021-04-27 04:21:39.002 DEBUG       yangsuite.ysyangtree.context : 500]
Successfully loaded openconfig-network-instance-l2
[2021-04-27 04:21:39.021 DEBUG       yangsuite.ysyangtree.context : 500]
Successfully loaded openconfig-packet-match
[2021-04-27 04:21:39.050 DEBUG       yangsuite.ysyangtree.context : 500]
Successfully loaded openconfig-platform
[2021-04-27 04:21:39.080 INFO    yangsuite.ysfilemanager.filemanager.yangset : 280]
Loaded YANG set 'CSR100v' owned by 'nsindiso' (543 modules) from /home/nsindiso/Documents/Model_Driven_Telemetry_Lab/[]/users/nsindiso/yangsets/csr100v
[2021-04-27 04:21:39.089 DEBUG       yangsuite.ysyangtree.context : 500]
Successfully loaded openconfig-routing-policy
[2021-04-27 04:21:39.110 DEBUG       yangsuite.ysyangtree.context : 500]
Successfully loaded openconfig-segment-routing
[2021-04-27 04:21:39.114 DEBUG       yangsuite.ysyangtree.context : 159]
Cached ctx has same yangset
[2021-04-27 04:21:39.116 INFO                       django.server : 124]
"GET /yangtree/gettree/?yangset=nsindiso%2Bcsr100v HTTP/1.1" 200 64
[2021-04-27 04:21:39.153 DEBUG       yangsuite.ysyangtree.context : 500]
Successfully loaded openconfig-system
[2021-04-27 04:21:39.155 DEBUG   yangsuite.ysfilemanager.filemanager.utility :  76]
Split userset "nsindiso+csr100v" to user "nsindiso", setname/reponame "csr100v"
[2021-04-27 04:21:39.170 DEBUG       yangsuite.ysyangtree.context : 500]
Successfully loaded openconfig-vlan
[2021-04-27 04:21:39.176 DEBUG       yangsuite.ysyangtree.context : 500]
Successfully loaded openconfig-yang-types
[2021-04-27 04:21:39.190 DEBUG   yangsuite.ysfilemanager.filemanager.utility :  76]
Split userset "nsindiso+csr100v" to user "nsindiso", setname/reponame "csr100v"
[2021-04-27 04:21:39.419 DEBUG       yangsuite.ysyangtree.context : 500]
Successfully loaded cisco-xe-openconfig-acl-deviation
[2021-04-27 04:21:39.422 DEBUG       yangsuite.ysyangtree.context : 500]
Successfully loaded cisco-xe-openconfig-aft-deviation
[2021-04-27 04:21:39.446 DEBUG       yangsuite.ysyangtree.context : 500]
Successfully loaded cisco-xe-openconfig-bgp-deviation
[2021-04-27 04:21:39.455 DEBUG       yangsuite.ysyangtree.context : 500]
Successfully loaded cisco-xe-openconfig-bgp-policy-deviation
[2021-04-27 04:21:39.461 DEBUG       yangsuite.ysyangtree.context : 500]
Successfully loaded cisco-xe-openconfig-if-ip-deviation
[2021-04-27 04:21:39.474 DEBUG       yangsuite.ysyangtree.context : 500]
Successfully loaded cisco-xe-openconfig-interfaces-deviation
[2021-04-27 04:21:39.481 DEBUG       yangsuite.ysyangtree.context : 500]
Successfully loaded cisco-xe-openconfig-isis-deviation
[2021-04-27 04:21:39.488 DEBUG       yangsuite.ysyangtree.context : 500]
Successfully loaded cisco-xe-openconfig-lldp-deviation
[2021-04-27 04:21:39.492 DEBUG       yangsuite.ysyangtree.context : 500]
Successfully loaded cisco-xe-openconfig-mpls-deviation
[2021-04-27 04:21:39.573 DEBUG       yangsuite.ysyangtree.context : 500]
Successfully loaded cisco-xe-openconfig-network-instance-deviation
[2021-04-27 04:21:39.588 DEBUG       yangsuite.ysyangtree.context : 500]
Successfully loaded cisco-xe-openconfig-routing-policy-deviation
[2021-04-27 04:21:39.597 DEBUG       yangsuite.ysyangtree.context : 500]
Successfully loaded cisco-xe-openconfig-segment-routing-deviation
[2021-04-27 04:21:39.608 DEBUG       yangsuite.ysyangtree.context : 500]
Successfully loaded cisco-xe-routing-csr-openconfig-platform-deviation
[2021-04-27 04:21:39.621 DEBUG       yangsuite.ysyangtree.context : 500]
Successfully loaded cisco-xe-routing-openconfig-system-deviation
[2021-04-27 04:21:39.628 DEBUG       yangsuite.ysyangtree.context : 500]
Successfully loaded cisco-xe-routing-openconfig-system-management-deviation
[2021-04-27 04:21:39.634 DEBUG       yangsuite.ysyangtree.context : 500]
Successfully loaded cisco-xe-routing-openconfig-vlan-deviation
[2021-04-27 04:21:39.635 DEBUG       yangsuite.ysyangtree.context : 518]
Have now loaded 94 modules, of 543 available in repository
[2021-04-27 04:21:39.648 DEBUG       yangsuite.ysyangtree.context : 433]
Found identity derivers of ietf-interfaces: ['iana-if-type']
[2021-04-27 04:21:39.651 DEBUG       yangsuite.ysyangtree.context : 433]
Found identity derivers of openconfig-if-ethernet: ['cisco-xe-openconfig-if-ethernet-ext']
[2021-04-27 04:21:39.654 DEBUG       yangsuite.ysyangtree.context : 433]
Found identity derivers of openconfig-packet-match-types: ['cisco-xe-openconfig-acl-ext']
[2021-04-27 04:21:39.656 DEBUG       yangsuite.ysyangtree.context : 433]
Found identity derivers of openconfig-aaa-types: ['openconfig-aaa-tacacs', 'openconfig-aaa-radius']
[2021-04-27 04:21:39.658 DEBUG       yangsuite.ysyangtree.context : 443]
Found 3 additional modules that derive identities: ['cisco-xe-openconfig-acl-ext', 'iana-if-type', 'openconfig-aaa-tacacs']
[2021-04-27 04:21:39.668 DEBUG       yangsuite.ysyangtree.context : 500]
Successfully loaded cisco-xe-openconfig-acl-ext
[2021-04-27 04:21:39.738 INFO    yangsuite.ysfilemanager.filemanager.yangset : 280]
Loaded YANG set 'CSR100v' owned by 'nsindiso' (543 modules) from /home/nsindiso/Documents/Model_Driven_Telemetry_Lab/[]/users/nsindiso/yangsets/csr100v
[2021-04-27 04:21:39.758 DEBUG       yangsuite.ysyangtree.context : 159]
Cached ctx has same yangset
[2021-04-27 04:21:39.761 INFO                       django.server : 124]
"GET /yangtree/gettree/?yangset=nsindiso%2Bcsr100v HTTP/1.1" 200 59
[2021-04-27 04:21:39.782 DEBUG       yangsuite.ysyangtree.context : 500]
Successfully loaded iana-if-type
[2021-04-27 04:21:39.790 DEBUG       yangsuite.ysyangtree.context : 500]
Successfully loaded openconfig-aaa-tacacs
[2021-04-27 04:21:39.796 DEBUG   yangsuite.ysfilemanager.filemanager.utility :  76]
Split userset "nsindiso+csr100v" to user "nsindiso", setname/reponame "csr100v"
[2021-04-27 04:21:39.796 DEBUG       yangsuite.ysyangtree.context : 518]
Have now loaded 95 modules, of 543 available in repository
[2021-04-27 04:21:39.803 DEBUG   yangsuite.ysfilemanager.filemanager.utility :  76]
Split userset "nsindiso+csr100v" to user "nsindiso", setname/reponame "csr100v"
[2021-04-27 04:21:39.804 INFO                       django.server : 124]
"POST /yangtree/gettree/ HTTP/1.1" 200 1458
[2021-04-27 04:21:39.833 INFO                       django.server : 124]
"GET /static/yangsuite/css/excite-bike/images/ui-icons_fcdd4a_256x240.png HTTP/1.1" 304 0
[2021-04-27 04:21:39.842 INFO                       django.server : 124]
"GET /static/yangsuite/css/excite-bike/images/ui-icons_0a82eb_256x240.png HTTP/1.1" 304 0
[2021-04-27 04:21:39.868 INFO                       django.server : 124]
"GET /static/yangsuite/css/excite-bike/images/ui-icons_ffffff_256x240.png HTTP/1.1" 304 0
[2021-04-27 04:21:39.894 INFO    yangsuite.ysfilemanager.filemanager.yangset : 280]
Loaded YANG set 'CSR100v' owned by 'nsindiso' (543 modules) from /home/nsindiso/Documents/Model_Driven_Telemetry_Lab/[]/users/nsindiso/yangsets/csr100v
[2021-04-27 04:21:39.895 DEBUG       yangsuite.ysyangtree.context : 159]
Cached ctx has same yangset
[2021-04-27 04:21:39.897 INFO                       django.server : 124]
"GET /yangtree/gettree/?yangset=nsindiso%2Bcsr100v HTTP/1.1" 200 62
[2021-04-27 04:21:39.908 DEBUG   yangsuite.ysfilemanager.filemanager.utility :  76]
Split userset "nsindiso+csr100v" to user "nsindiso", setname/reponame "csr100v"
[2021-04-27 04:21:39.912 INFO                       django.server : 124]
"GET /static/yangsuite/js/jstree/themes/default/throbber.gif HTTP/1.1" 304 0
[2021-04-27 04:21:39.913 DEBUG   yangsuite.ysfilemanager.filemanager.utility :  76]
Split userset "nsindiso+csr100v" to user "nsindiso", setname/reponame "csr100v"
[2021-04-27 04:21:39.965 INFO    yangsuite.ysfilemanager.filemanager.yangset : 280]
Loaded YANG set 'CSR100v' owned by 'nsindiso' (543 modules) from /home/nsindiso/Documents/Model_Driven_Telemetry_Lab/[]/users/nsindiso/yangsets/csr100v
[2021-04-27 04:21:39.967 DEBUG       yangsuite.ysyangtree.context : 159]
Cached ctx has same yangset
[2021-04-27 04:21:39.968 INFO                       django.server : 124]
"GET /yangtree/gettree/?yangset=nsindiso%2Bcsr100v HTTP/1.1" 200 62
[2021-04-27 04:21:40.111 INFO                       django.server : 124]
"GET /static/yangsuite/js/jstree/themes/default/32px.png HTTP/1.1" 304 0
[2021-04-27 09:14:58.713 DEBUG                     yangsuite.urls :  90]
Including ysyangtree.urls at 'yangtree'
[2021-04-27 09:14:58.730 DEBUG                     yangsuite.urls :  90]
Including ysnetconf.urls at 'netconf'
[2021-04-27 09:14:59.061 DEBUG                     yangsuite.urls :  90]
Including ysfilemanager.urls at 'filemanager'
[2021-04-27 09:14:59.064 DEBUG                     yangsuite.urls :  90]
Including ysdevices.urls at 'devices'
[2021-04-27 09:15:04.920 DEBUG                     yangsuite.urls :  90]
Including ysyangtree.urls at 'yangtree'
[2021-04-27 09:15:04.930 DEBUG                     yangsuite.urls :  90]
Including ysnetconf.urls at 'netconf'
[2021-04-27 09:15:05.127 DEBUG                     yangsuite.urls :  90]
Including ysfilemanager.urls at 'filemanager'
[2021-04-27 09:15:05.139 DEBUG                     yangsuite.urls :  90]
Including ysdevices.urls at 'devices'
[2021-04-27 09:15:28.417 INFO                       django.server : 124]
"GET / HTTP/1.1" 302 0
[2021-04-27 09:15:28.448 INFO                       django.server : 124]
"GET /help/ HTTP/1.1" 302 0
[2021-04-27 09:15:28.505 DEBUG                    yangsuite.views : 362]
Sorted pages based on page_order ['introduction']: []
[2021-04-27 09:15:28.531 DEBUG                    yangsuite.views : 362]
Sorted pages based on page_order ['index']: [{'title': 'Exploring YANG Models', 'content': 'explore'}]
[2021-04-27 09:15:28.547 DEBUG                    yangsuite.views : 362]
Sorted pages based on page_order ['rpcs', 'notifications', 'tasks', 'lock-unlock']: [{'title': 'Using NETCONF RPCs', 'content': 'rpcs'}, {'title': 'Working with NETCONF Notification Streams', 'content': 'notifications'}, {'title': 'Using Replays for repeated workflows', 'content': 'tasks'}, {'title': 'Locking and unlocking datastores', 'content': 'lock-unlock'}]
[2021-04-27 09:15:28.565 DEBUG                    yangsuite.views : 362]
Sorted pages based on page_order ['create_repository', 'upload_schemas', 'download_schemas', 'scp_schemas', 'git_schemas', 'define_yangset']: [{'title': 'Constructing and populating a YANG module repository', 'content': 'create_repository'}, {'title': 'Uploading YANG files from the local filesystem to a YANG repository', 'content': 'upload_schemas'}, {'title': 'Downloading YANG files via NETCONF from a device to a YANG repository', 'content': 'download_schemas'}, {'title': 'Copying YANG modules via SCP from a server to a YANG repository', 'content': 'scp_schemas'}, {'title': 'Importing YANG modules from a Git repository to a YANG repository', 'content': 'git_schemas'}, {'title': 'Defining a YANG module set', 'content': 'define_yangset'}]
[2021-04-27 09:15:28.573 DEBUG                    yangsuite.views : 362]
Sorted pages based on page_order ['index']: [{'title': 'Managing device profiles', 'content': 'devices'}]
[2021-04-27 09:15:28.607 DEBUG       yangsuite.context_processors : 136]
Adding menus from yangsuite
[2021-04-27 09:15:28.611 DEBUG       yangsuite.context_processors : 172]
Added menu 'Admin'; menus are now ('Admin',)
[2021-04-27 09:15:28.611 DEBUG       yangsuite.context_processors : 186]
Added menu item 'Manage users → /admin' in menu 'Admin'
[2021-04-27 09:15:28.612 DEBUG       yangsuite.context_processors : 186]
Added menu item 'Manage plugins → /yangsuite/plugins/' in menu 'Admin'
[2021-04-27 09:15:28.612 DEBUG       yangsuite.context_processors : 186]
Added menu item 'View logs → /yangsuite/logs' in menu 'Admin'
[2021-04-27 09:15:28.612 DEBUG       yangsuite.context_processors : 172]
Added menu 'Setup'; menus are now ('Admin', 'Setup')
[2021-04-27 09:15:28.613 DEBUG       yangsuite.context_processors : 136]
Adding menus from ysfilemanager
[2021-04-27 09:15:28.613 DEBUG       yangsuite.context_processors : 186]
Added menu item 'YANG files and repositories → /filemanager/repository/manage/' in menu 'Setup'
[2021-04-27 09:15:28.614 DEBUG       yangsuite.context_processors : 186]
Added menu item 'YANG module sets → /filemanager/yangsets/manage/' in menu 'Setup'
[2021-04-27 09:15:28.614 DEBUG       yangsuite.context_processors : 136]
Adding menus from ysdevices
[2021-04-27 09:15:28.614 DEBUG       yangsuite.context_processors : 186]
Added menu item 'Device profiles → /devices/devprofile' in menu 'Setup'
[2021-04-27 09:15:28.615 DEBUG       yangsuite.context_processors : 136]
Adding menus from ysyangtree
[2021-04-27 09:15:28.615 DEBUG       yangsuite.context_processors : 172]
Added menu 'Explore'; menus are now ('Admin', 'Setup', 'Explore')
[2021-04-27 09:15:28.616 DEBUG       yangsuite.context_processors : 186]
Added menu item 'YANG → /yangtree/explore' in menu 'Explore'
[2021-04-27 09:15:28.616 DEBUG       yangsuite.context_processors : 136]
Adding menus from ysnetconf
[2021-04-27 09:15:28.617 DEBUG       yangsuite.context_processors : 172]
Added menu 'Protocols'; menus are now ('Admin', 'Setup', 'Explore', 'Protocols')
[2021-04-27 09:15:28.617 DEBUG       yangsuite.context_processors : 186]
Added menu item 'NETCONF → /netconf/getyang' in menu 'Protocols'
[2021-04-27 09:15:28.621 INFO        yangsuite.context_processors : 121]
yangsuite_menus is now: OrderedDict([('Admin', [YSMenuItem(text='Manage users', url='/admin'), YSMenuItem(text='Manage plugins', url='/yangsuite/plugins/'), YSMenuItem(text='View logs', url='/yangsuite/logs')]), ('Setup', [YSMenuItem(text='YANG files and repositories', url='/filemanager/repository/manage/'), YSMenuItem(text='YANG module sets', url='/filemanager/yangsets/manage/'), YSMenuItem(text='Device profiles', url='/devices/devprofile')]), ('Explore', [YSMenuItem(text='YANG', url='/yangtree/explore')]), ('Protocols', [YSMenuItem(text='NETCONF', url='/netconf/getyang')])])
[2021-04-27 09:15:28.651 INFO                       django.server : 124]
"GET /help/yangsuite/ HTTP/1.1" 200 13750
[2021-04-27 09:15:28.811 INFO                       django.server : 124]
"GET /static/yangsuite/js/jquery-1.11.2.min.js HTTP/1.1" 304 0
[2021-04-27 09:15:28.833 INFO                       django.server : 124]
"GET /static/yangsuite/js/jstree/themes/default/style.min.css HTTP/1.1" 304 0
[2021-04-27 09:15:28.845 INFO                       django.server : 124]
"GET /static/yangsuite/css/excite-bike/jquery-ui.min.css HTTP/1.1" 304 0
[2021-04-27 09:15:28.857 INFO                       django.server : 124]
"GET /static/yangsuite/css/cui-standard-1.3.3.min.css HTTP/1.1" 304 0
[2021-04-27 09:15:28.866 INFO                       django.server : 124]
"GET /static/yangsuite/js/jstree/jstreegrid.js HTTP/1.1" 304 0
[2021-04-27 09:15:28.868 INFO                       django.server : 124]
"GET /static/yangsuite/js/jstree/jstree.js HTTP/1.1" 304 0
[2021-04-27 09:15:28.869 INFO                       django.server : 124]
"GET /static/yangsuite/js/jquery-ui.min.js HTTP/1.1" 304 0
[2021-04-27 09:15:28.880 INFO                       django.server : 124]
"GET /static/yangsuite/js/js.cookie.js HTTP/1.1" 304 0
[2021-04-27 09:15:28.890 INFO                       django.server : 124]
"GET /static/yangsuite/js/yangsuite.js HTTP/1.1" 304 0
[2021-04-27 09:15:28.892 INFO                       django.server : 124]
"GET /static/yangsuite/css/yangsuite-cui.css HTTP/1.1" 304 0
[2021-04-27 09:15:28.955 INFO                       django.server : 124]
"GET /static/yangsuite/favicon/yangsuite.png HTTP/1.1" 304 0
[2021-04-27 09:15:29.079 INFO                       django.server : 124]
"POST /gtm HTTP/1.1" 200 20
[2021-04-27 09:15:29.479 INFO                       django.server : 124]
"GET /yangsuite/plugins/mode HTTP/1.1" 200 15
[2021-04-27 09:15:29.511 INFO                       django.server : 124]
"GET /static/yangsuite/fonts/CiscoSans/CiscoSansTTRegular.woff2 HTTP/1.1" 304 0
[2021-04-27 09:15:29.562 INFO                       django.server : 124]
"GET /static/yangsuite/fonts/CiscoSans/CiscoSansTTLight.woff2 HTTP/1.1" 304 0
[2021-04-27 09:15:29.585 INFO                       django.server : 124]
"GET /static/yangsuite/fonts/cui-font-1.3.3.woff2 HTTP/1.1" 304 0
[2021-04-27 09:15:29.588 INFO                       django.server : 124]
"GET /static/yangsuite/fonts/CiscoSans/CiscoSansTTExtraLight.woff2 HTTP/1.1" 304 0
[2021-04-27 09:15:29.599 INFO                       django.server : 124]
"GET /static/yangsuite/fonts/CiscoSans/CiscoSansTTThin.woff2 HTTP/1.1" 200 51904
[2021-04-27 09:15:29.603 INFO                       django.server : 124]
"GET /static/yangsuite/fonts/CiscoSans/CiscoSansTTBold.woff2 HTTP/1.1" 304 0
[2021-04-27 09:15:29.959 INFO                       django.server : 124]
"GET /static/yangsuite/favicon/manifest.json HTTP/1.1" 304 0
[2021-04-27 09:15:33.976 INFO                       django.server : 124]
"GET /yangtree/explore/ HTTP/1.1" 200 19815
[2021-04-27 09:15:34.046 INFO                       django.server : 124]
"GET /static/ysfilemanager/js/yangsets.js HTTP/1.1" 304 0
[2021-04-27 09:15:34.063 INFO                       django.server : 124]
"GET /static/ysyangtree/js/yangtree.js HTTP/1.1" 304 0
[2021-04-27 09:15:34.099 INFO                       django.server : 124]
"GET /static/ysyangtree/js/tasks.js HTTP/1.1" 304 0
[2021-04-27 09:15:34.102 INFO                       django.server : 124]
"GET /static/ysyangtree/css/chosen/chosen.css HTTP/1.1" 304 0
[2021-04-27 09:15:34.104 INFO                       django.server : 124]
"GET /static/ysyangtree/css/yangtree.css HTTP/1.1" 304 0
[2021-04-27 09:15:34.109 INFO                       django.server : 124]
"GET /static/ysyangtree/js/chosen/chosen.jquery.js HTTP/1.1" 304 0
[2021-04-27 09:15:34.210 INFO                       django.server : 124]
"POST /gtm HTTP/1.1" 200 20
[2021-04-27 09:15:34.220 INFO                       django.server : 124]
"GET /static/yangsuite/js/fontawesome-all.js HTTP/1.1" 304 0
[2021-04-27 09:15:34.559 INFO                       django.server : 124]
"GET /yangsuite/plugins/mode HTTP/1.1" 200 15
[2021-04-27 09:15:34.661 INFO                       django.server : 124]
"GET /filemanager/yangsets/list/?readonly=true HTTP/1.1" 200 77
[2021-04-27 09:15:38.144 DEBUG   yangsuite.ysfilemanager.filemanager.utility :  76]
Split userset "nsindiso+csr100v" to user "nsindiso", setname/reponame "csr100v"
[2021-04-27 09:15:38.148 DEBUG   yangsuite.ysfilemanager.filemanager.utility :  76]
Split userset "nsindiso+csr100v" to user "nsindiso", setname/reponame "csr100v"
[2021-04-27 09:15:38.254 INFO    yangsuite.ysfilemanager.filemanager.yangset : 280]
Loaded YANG set 'CSR100v' owned by 'nsindiso' (543 modules) from /home/nsindiso/Documents/Model_Driven_Telemetry_Lab/[]/users/nsindiso/yangsets/csr100v
[2021-04-27 09:15:38.261 INFO                       django.server : 124]
"GET /filemanager/yangsets/get/?yangset=nsindiso%2Bcsr100v HTTP/1.1" 200 33088
[2021-04-27 09:16:08.151 INFO                       django.server : 124]
"GET /static/ysyangtree/css/chosen/chosen-sprite.png HTTP/1.1" 304 0
[2021-04-27 09:16:10.842 DEBUG   yangsuite.ysfilemanager.filemanager.utility :  76]
Split userset "nsindiso+csr100v" to user "nsindiso", setname/reponame "csr100v"
[2021-04-27 09:16:10.845 DEBUG   yangsuite.ysfilemanager.filemanager.utility :  76]
Split userset "nsindiso+csr100v" to user "nsindiso", setname/reponame "csr100v"
[2021-04-27 09:16:10.948 INFO    yangsuite.ysfilemanager.filemanager.yangset : 280]
Loaded YANG set 'CSR100v' owned by 'nsindiso' (543 modules) from /home/nsindiso/Documents/Model_Driven_Telemetry_Lab/[]/users/nsindiso/yangsets/csr100v
[2021-04-27 09:16:10.949 DEBUG   yangsuite.ysfilemanager.filemanager.utility :  76]
Split userset "nsindiso+csr100v" to user "nsindiso", setname/reponame "csr100v"
[2021-04-27 09:16:10.954 DEBUG   yangsuite.ysfilemanager.filemanager.utility :  76]
Split userset "nsindiso+csr100v" to user "nsindiso", setname/reponame "csr100v"
[2021-04-27 09:16:11.040 INFO    yangsuite.ysfilemanager.filemanager.yangset : 280]
Loaded YANG set 'CSR100v' owned by 'nsindiso' (543 modules) from /home/nsindiso/Documents/Model_Driven_Telemetry_Lab/[]/users/nsindiso/yangsets/csr100v
[2021-04-27 09:16:11.041 INFO        yangsuite.ysyangtree.context : 288]
Initializing context for path 'YANG set 'CSR100v' owned by 'nsindiso' (543 modules)'
[2021-04-27 09:16:11.042 INFO        yangsuite.ysyangtree.context : 296]
Context stored with reference 'nsindiso'
[2021-04-27 09:16:11.044 DEBUG       yangsuite.ysyangtree.context : 396]
No module loading requested.
[2021-04-27 09:16:11.044 INFO        yangsuite.ysyangtree.context : 340]
Context 'nsindiso+csr100v' now fully loaded and ready
[2021-04-27 09:16:11.842 DEBUG   yangsuite.ysfilemanager.filemanager.utility :  76]
Split userset "nsindiso+csr100v" to user "nsindiso", setname/reponame "csr100v"
[2021-04-27 09:16:11.847 DEBUG   yangsuite.ysfilemanager.filemanager.utility :  76]
Split userset "nsindiso+csr100v" to user "nsindiso", setname/reponame "csr100v"
[2021-04-27 09:16:12.089 INFO    yangsuite.ysfilemanager.filemanager.yangset : 280]
Loaded YANG set 'CSR100v' owned by 'nsindiso' (543 modules) from /home/nsindiso/Documents/Model_Driven_Telemetry_Lab/[]/users/nsindiso/yangsets/csr100v
[2021-04-27 09:16:12.100 DEBUG       yangsuite.ysyangtree.context : 159]
Cached ctx has same yangset
[2021-04-27 09:16:12.107 INFO                       django.server : 124]
"GET /yangtree/gettree/?yangset=nsindiso%2Bcsr100v HTTP/1.1" 200 39
[2021-04-27 09:16:12.841 DEBUG   yangsuite.ysfilemanager.filemanager.utility :  76]
Split userset "nsindiso+csr100v" to user "nsindiso", setname/reponame "csr100v"
[2021-04-27 09:16:12.845 DEBUG   yangsuite.ysfilemanager.filemanager.utility :  76]
Split userset "nsindiso+csr100v" to user "nsindiso", setname/reponame "csr100v"
[2021-04-27 09:16:13.075 INFO    yangsuite.ysfilemanager.filemanager.yangset : 280]
Loaded YANG set 'CSR100v' owned by 'nsindiso' (543 modules) from /home/nsindiso/Documents/Model_Driven_Telemetry_Lab/[]/users/nsindiso/yangsets/csr100v
[2021-04-27 09:16:13.087 DEBUG       yangsuite.ysyangtree.context : 159]
Cached ctx has same yangset
[2021-04-27 09:16:13.093 INFO                       django.server : 124]
"GET /yangtree/gettree/?yangset=nsindiso%2Bcsr100v HTTP/1.1" 200 39
[2021-04-27 09:16:13.276 DEBUG       yangsuite.ysyangtree.context : 400]
Loading module(s) {'Cisco-IOS-XE-dhcp-oper'} from repository YANG set 'CSR100v' owned by 'nsindiso' (543 modules)
[2021-04-27 09:16:13.279 DEBUG       yangsuite.ysyangtree.context : 407]
Will load upstream modules as well: []
[2021-04-27 09:16:13.499 DEBUG       yangsuite.ysyangtree.context : 500]
Successfully loaded Cisco-IOS-XE-dhcp-oper
[2021-04-27 09:16:13.500 DEBUG       yangsuite.ysyangtree.context : 518]
Have now loaded 4 modules, of 543 available in repository
[2021-04-27 09:16:13.525 INFO                       django.server : 124]
"POST /yangtree/gettree/ HTTP/1.1" 200 85034
[2021-04-27 09:16:13.596 INFO                       django.server : 124]
"GET /static/yangsuite/js/jstree/themes/default/throbber.gif HTTP/1.1" 304 0
[2021-04-27 09:16:13.925 INFO                       django.server : 124]
"GET /static/yangsuite/js/jstree/themes/default/32px.png HTTP/1.1" 304 0
[2021-04-27 09:16:25.902 INFO                       django.server : 124]
"POST /yangtree/getrfc/ HTTP/1.1" 200 7895
[2021-04-27 09:17:17.050 DEBUG   yangsuite.ysfilemanager.filemanager.utility :  76]
Split userset "nsindiso+csr100v" to user "nsindiso", setname/reponame "csr100v"
[2021-04-27 09:17:17.053 DEBUG   yangsuite.ysfilemanager.filemanager.utility :  76]
Split userset "nsindiso+csr100v" to user "nsindiso", setname/reponame "csr100v"
[2021-04-27 09:17:17.198 INFO    yangsuite.ysfilemanager.filemanager.yangset : 280]
Loaded YANG set 'CSR100v' owned by 'nsindiso' (543 modules) from /home/nsindiso/Documents/Model_Driven_Telemetry_Lab/[]/users/nsindiso/yangsets/csr100v
[2021-04-27 09:17:17.199 DEBUG   yangsuite.ysfilemanager.filemanager.utility :  76]
Split userset "nsindiso+csr100v" to user "nsindiso", setname/reponame "csr100v"
[2021-04-27 09:17:17.200 DEBUG   yangsuite.ysfilemanager.filemanager.utility :  76]
Split userset "nsindiso+csr100v" to user "nsindiso", setname/reponame "csr100v"
[2021-04-27 09:17:17.307 INFO    yangsuite.ysfilemanager.filemanager.yangset : 280]
Loaded YANG set 'CSR100v' owned by 'nsindiso' (543 modules) from /home/nsindiso/Documents/Model_Driven_Telemetry_Lab/[]/users/nsindiso/yangsets/csr100v
[2021-04-27 09:17:17.310 DEBUG       yangsuite.ysyangtree.context : 159]
Cached ctx has same yangset
[2021-04-27 09:17:17.311 DEBUG       yangsuite.ysyangtree.context : 400]
Loading module(s) {'Cisco-IOS-XE-install-oper'} from repository YANG set 'CSR100v' owned by 'nsindiso' (543 modules)
[2021-04-27 09:17:17.312 DEBUG       yangsuite.ysyangtree.context : 407]
Will load upstream modules as well: []
[2021-04-27 09:17:17.848 DEBUG       yangsuite.ysyangtree.context : 500]
Successfully loaded Cisco-IOS-XE-install-oper
[2021-04-27 09:17:17.850 DEBUG       yangsuite.ysyangtree.context : 518]
Have now loaded 6 modules, of 543 available in repository
[2021-04-27 09:17:17.951 INFO        yangsuite.ysyangtree.ymodels :  88]
Replacing cached instance YSYangModels (YANG models ('Cisco-IOS-XE-dhcp-oper',), context YSContext (repository YANG set 'CSR100v' owned by 'nsindiso' (543 modules)) with 6 modules loaded) with YSYangModels (YANG models ('Cisco-IOS-XE-install-oper',), context YSContext (repository YANG set 'CSR100v' owned by 'nsindiso' (543 modules)) with 6 modules loaded) under key "nsindiso"
[2021-04-27 09:17:17.979 INFO                       django.server : 124]
"POST /yangtree/gettree/ HTTP/1.1" 200 612686
[2021-04-27 09:17:44.772 INFO                       django.server : 124]
"POST /yangtree/getrfc/ HTTP/1.1" 200 7895
[2021-04-27 09:17:50.341 INFO                       django.server : 124]
"POST /yangtree/getrfc/ HTTP/1.1" 200 15944
[2021-04-27 09:18:20.363 DEBUG   yangsuite.ysfilemanager.filemanager.utility :  76]
Split userset "nsindiso+csr100v" to user "nsindiso", setname/reponame "csr100v"
[2021-04-27 09:18:20.370 DEBUG   yangsuite.ysfilemanager.filemanager.utility :  76]
Split userset "nsindiso+csr100v" to user "nsindiso", setname/reponame "csr100v"
[2021-04-27 09:18:20.479 INFO    yangsuite.ysfilemanager.filemanager.yangset : 280]
Loaded YANG set 'CSR100v' owned by 'nsindiso' (543 modules) from /home/nsindiso/Documents/Model_Driven_Telemetry_Lab/[]/users/nsindiso/yangsets/csr100v
[2021-04-27 09:18:20.480 DEBUG   yangsuite.ysfilemanager.filemanager.utility :  76]
Split userset "nsindiso+csr100v" to user "nsindiso", setname/reponame "csr100v"
[2021-04-27 09:18:20.483 DEBUG   yangsuite.ysfilemanager.filemanager.utility :  76]
Split userset "nsindiso+csr100v" to user "nsindiso", setname/reponame "csr100v"
[2021-04-27 09:18:20.570 INFO    yangsuite.ysfilemanager.filemanager.yangset : 280]
Loaded YANG set 'CSR100v' owned by 'nsindiso' (543 modules) from /home/nsindiso/Documents/Model_Driven_Telemetry_Lab/[]/users/nsindiso/yangsets/csr100v
[2021-04-27 09:18:20.583 DEBUG       yangsuite.ysyangtree.context : 159]
Cached ctx has same yangset
[2021-04-27 09:18:20.585 DEBUG       yangsuite.ysyangtree.context : 400]
Loading module(s) {'Cisco-IOS-XE-bgp-oper'} from repository YANG set 'CSR100v' owned by 'nsindiso' (543 modules)
[2021-04-27 09:18:20.587 DEBUG       yangsuite.ysyangtree.context : 407]
Will load upstream modules as well: []
[2021-04-27 09:18:21.116 DEBUG       yangsuite.ysyangtree.context : 500]
Successfully loaded Cisco-IOS-XE-bgp-oper
[2021-04-27 09:18:21.117 DEBUG       yangsuite.ysyangtree.context : 518]
Have now loaded 9 modules, of 543 available in repository
[2021-04-27 09:18:21.185 INFO        yangsuite.ysyangtree.ymodels :  88]
Replacing cached instance YSYangModels (YANG models ('Cisco-IOS-XE-install-oper',), context YSContext (repository YANG set 'CSR100v' owned by 'nsindiso' (543 modules)) with 9 modules loaded) with YSYangModels (YANG models ('Cisco-IOS-XE-bgp-oper',), context YSContext (repository YANG set 'CSR100v' owned by 'nsindiso' (543 modules)) with 9 modules loaded) under key "nsindiso"
[2021-04-27 09:18:21.204 INFO                       django.server : 124]
"POST /yangtree/gettree/ HTTP/1.1" 200 416773
[2021-04-27 09:18:55.100 INFO                       django.server : 124]
"POST /yangtree/getrfc/ HTTP/1.1" 200 7895
[2021-04-27 09:19:05.856 INFO                       django.server : 124]
"POST /yangtree/getrfc/ HTTP/1.1" 200 7895
[2021-04-27 09:39:50.141 DEBUG   yangsuite.ysfilemanager.filemanager.utility :  76]
Split userset "nsindiso+csr100v" to user "nsindiso", setname/reponame "csr100v"
[2021-04-27 09:39:50.144 DEBUG   yangsuite.ysfilemanager.filemanager.utility :  76]
Split userset "nsindiso+csr100v" to user "nsindiso", setname/reponame "csr100v"
[2021-04-27 09:39:50.353 INFO    yangsuite.ysfilemanager.filemanager.yangset : 280]
Loaded YANG set 'CSR100v' owned by 'nsindiso' (543 modules) from /home/nsindiso/Documents/Model_Driven_Telemetry_Lab/[]/users/nsindiso/yangsets/csr100v
[2021-04-27 09:39:50.374 INFO                       django.server : 124]
"POST /yangtree/gettree/ HTTP/1.1" 200 416773
[2021-04-27 09:39:57.668 INFO                       django.server : 124]
"POST /yangtree/getrfc/ HTTP/1.1" 200 7895
[2021-04-27 09:40:01.981 INFO                       django.server : 124]
"POST /yangtree/getrfc/ HTTP/1.1" 200 7895
[2021-04-27 09:40:04.398 INFO                       django.server : 124]
"POST /yangtree/getrfc/ HTTP/1.1" 200 7895
[2021-04-27 09:40:05.728 INFO                       django.server : 124]
"POST /yangtree/getrfc/ HTTP/1.1" 200 7895
[2021-04-27 20:55:18.766 INFO                       django.server : 124]
"GET /devices/devprofile HTTP/1.1" 200 13552
[2021-04-27 20:55:18.950 INFO                       django.server : 124]
"GET /static/yangsuite/js/jquery-1.11.2.min.js HTTP/1.1" 304 0
[2021-04-27 20:55:18.971 INFO                       django.server : 124]
"GET /static/yangsuite/js/jquery-ui.min.js HTTP/1.1" 304 0
[2021-04-27 20:55:18.980 INFO                       django.server : 124]
"GET /static/yangsuite/js/jstree/jstree.js HTTP/1.1" 304 0
[2021-04-27 20:55:19.013 INFO                       django.server : 124]
"GET /static/yangsuite/js/js.cookie.js HTTP/1.1" 304 0
[2021-04-27 20:55:19.014 INFO                       django.server : 124]
"GET /static/yangsuite/js/jstree/jstreegrid.js HTTP/1.1" 304 0
[2021-04-27 20:55:19.032 INFO                       django.server : 124]
"GET /static/yangsuite/js/jstree/themes/default/style.min.css HTTP/1.1" 304 0
[2021-04-27 20:55:19.033 INFO                       django.server : 124]
"GET /static/yangsuite/css/cui-standard-1.3.3.min.css HTTP/1.1" 304 0
[2021-04-27 20:55:19.041 INFO                       django.server : 124]
"GET /static/yangsuite/css/excite-bike/jquery-ui.min.css HTTP/1.1" 304 0
[2021-04-27 20:55:19.045 INFO                       django.server : 124]
"GET /static/yangsuite/js/yangsuite.js HTTP/1.1" 304 0
[2021-04-27 20:55:19.096 INFO                       django.server : 124]
"GET /static/yangsuite/css/yangsuite-cui.css HTTP/1.1" 304 0
[2021-04-27 20:55:19.115 INFO                       django.server : 124]
"GET /static/ysnetconf/js/netconf.js HTTP/1.1" 304 0
[2021-04-27 20:55:19.136 INFO                       django.server : 124]
"GET /static/ysyangtree/js/yangtree.js HTTP/1.1" 304 0
[2021-04-27 20:55:19.140 INFO                       django.server : 124]
"GET /static/ysdevices/css/device.css HTTP/1.1" 304 0
[2021-04-27 20:55:19.150 INFO                       django.server : 124]
"GET /static/ysdevices/js/devices.js HTTP/1.1" 304 0
[2021-04-27 20:55:19.477 INFO                       django.server : 124]
"POST /gtm HTTP/1.1" 200 20
[2021-04-27 20:55:19.497 INFO                       django.server : 124]
"GET /static/yangsuite/favicon/yangsuite.png HTTP/1.1" 304 0
[2021-04-27 20:55:19.631 INFO                       django.server : 124]
"GET /static/yangsuite/fonts/CiscoSans/CiscoSansTTLight.woff2 HTTP/1.1" 304 0
[2021-04-27 20:55:19.639 INFO                       django.server : 124]
"GET /static/yangsuite/fonts/CiscoSans/CiscoSansTTRegular.woff2 HTTP/1.1" 304 0
[2021-04-27 20:55:19.640 INFO                       django.server : 124]
"GET /static/yangsuite/fonts/cui-font-1.3.3.woff2 HTTP/1.1" 304 0
[2021-04-27 20:55:19.650 INFO                       django.server : 124]
"GET /static/yangsuite/fonts/CiscoSans/CiscoSansTTExtraLight.woff2 HTTP/1.1" 304 0
[2021-04-27 20:55:19.820 DEBUG     yangsuite.ysdevices.devprofile : 181]
Device profile list (filter: None): [{'name': 'CSR100V', 'key': 'csr100v', 'description': 'MDT'}]
[2021-04-27 20:55:19.850 INFO                       django.server : 124]
"GET /devices/list/ HTTP/1.1" 200 74
[2021-04-27 20:55:19.856 INFO                       django.server : 124]
"GET /yangsuite/plugins/mode HTTP/1.1" 200 15
[2021-04-27 20:55:20.378 INFO                       django.server : 124]
"GET /static/yangsuite/favicon/manifest.json HTTP/1.1" 304 0
[2021-04-27 20:55:23.757 INFO                       django.server : 124]
"POST /netconf/session/start_end/ HTTP/1.1" 200 15
[2021-04-27 20:55:24.723 DEBUG                    django.template : 878]
Exception while resolving variable 'min' in template 'ysdevices/device_form.html'.
Traceback (most recent call last):
  File "/home/nsindiso/.local/lib/python3.6/site-packages/django/template/base.py", line 835, in _resolve_lookup
    current = current[bit]
KeyError: 'min'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/nsindiso/.local/lib/python3.6/site-packages/django/template/base.py", line 843, in _resolve_lookup
    current = getattr(current, bit)
AttributeError: 'dict' object has no attribute 'min'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/nsindiso/.local/lib/python3.6/site-packages/django/template/base.py", line 849, in _resolve_lookup
    current = current[int(bit)]
ValueError: invalid literal for int() with base 10: 'min'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/nsindiso/.local/lib/python3.6/site-packages/django/template/base.py", line 856, in _resolve_lookup
    (bit, current))  # missing attribute
django.template.base.VariableDoesNotExist: Failed lookup for key [min] in {'label': 'Profile Name', 'type': 'string', 'description': 'Name of this device profile', 'minLength': 1, 'required': True, 'default': 'CSR100V'}
[2021-04-27 20:55:24.739 DEBUG                    django.template : 878]
Exception while resolving variable 'max' in template 'ysdevices/device_form.html'.
Traceback (most recent call last):
  File "/home/nsindiso/.local/lib/python3.6/site-packages/django/template/base.py", line 835, in _resolve_lookup
    current = current[bit]
KeyError: 'max'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/nsindiso/.local/lib/python3.6/site-packages/django/template/base.py", line 843, in _resolve_lookup
    current = getattr(current, bit)
AttributeError: 'dict' object has no attribute 'max'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/nsindiso/.local/lib/python3.6/site-packages/django/template/base.py", line 849, in _resolve_lookup
    current = current[int(bit)]
ValueError: invalid literal for int() with base 10: 'max'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/nsindiso/.local/lib/python3.6/site-packages/django/template/base.py", line 856, in _resolve_lookup
    (bit, current))  # missing attribute
django.template.base.VariableDoesNotExist: Failed lookup for key [max] in {'label': 'Profile Name', 'type': 'string', 'description': 'Name of this device profile', 'minLength': 1, 'required': True, 'default': 'CSR100V'}
[2021-04-27 20:55:24.741 DEBUG                    django.template : 878]
Exception while resolving variable 'maxLength' in template 'ysdevices/device_form.html'.
Traceback (most recent call last):
  File "/home/nsindiso/.local/lib/python3.6/site-packages/django/template/base.py", line 835, in _resolve_lookup
    current = current[bit]
KeyError: 'maxLength'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/nsindiso/.local/lib/python3.6/site-packages/django/template/base.py", line 843, in _resolve_lookup
    current = getattr(current, bit)
AttributeError: 'dict' object has no attribute 'maxLength'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/nsindiso/.local/lib/python3.6/site-packages/django/template/base.py", line 849, in _resolve_lookup
    current = current[int(bit)]
ValueError: invalid literal for int() with base 10: 'maxLength'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/nsindiso/.local/lib/python3.6/site-packages/django/template/base.py", line 856, in _resolve_lookup
    (bit, current))  # missing attribute
django.template.base.VariableDoesNotExist: Failed lookup for key [maxLength] in {'label': 'Profile Name', 'type': 'string', 'description': 'Name of this device profile', 'minLength': 1, 'required': True, 'default': 'CSR100V'}
[2021-04-27 20:55:24.742 DEBUG                    django.template : 878]
Exception while resolving variable 'step' in template 'ysdevices/device_form.html'.
Traceback (most recent call last):
  File "/home/nsindiso/.local/lib/python3.6/site-packages/django/template/base.py", line 835, in _resolve_lookup
    current = current[bit]
KeyError: 'step'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/nsindiso/.local/lib/python3.6/site-packages/django/template/base.py", line 843, in _resolve_lookup
    current = getattr(current, bit)
AttributeError: 'dict' object has no attribute 'step'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/nsindiso/.local/lib/python3.6/site-packages/django/template/base.py", line 849, in _resolve_lookup
    current = current[int(bit)]
ValueError: invalid literal for int() with base 10: 'step'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/nsindiso/.local/lib/python3.6/site-packages/django/template/base.py", line 856, in _resolve_lookup
    (bit, current))  # missing attribute
django.template.base.VariableDoesNotExist: Failed lookup for key [step] in {'label': 'Profile Name', 'type': 'string', 'description': 'Name of this device profile', 'minLength': 1, 'required': True, 'default': 'CSR100V'}
[2021-04-27 20:55:24.744 DEBUG                    django.template : 878]
Exception while resolving variable 'min' in template 'ysdevices/device_form.html'.
Traceback (most recent call last):
  File "/home/nsindiso/.local/lib/python3.6/site-packages/django/template/base.py", line 835, in _resolve_lookup
    current = current[bit]
KeyError: 'min'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/nsindiso/.local/lib/python3.6/site-packages/django/template/base.py", line 843, in _resolve_lookup
    current = getattr(current, bit)
AttributeError: 'dict' object has no attribute 'min'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/nsindiso/.local/lib/python3.6/site-packages/django/template/base.py", line 849, in _resolve_lookup
    current = current[int(bit)]
ValueError: invalid literal for int() with base 10: 'min'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/nsindiso/.local/lib/python3.6/site-packages/django/template/base.py", line 856, in _resolve_lookup
    (bit, current))  # missing attribute
django.template.base.VariableDoesNotExist: Failed lookup for key [min] in {'type': 'string', 'description': 'Verbose description', 'default': 'MDT'}
[2021-04-27 20:55:24.745 DEBUG                    django.template : 878]
Exception while resolving variable 'minLength' in template 'ysdevices/device_form.html'.
Traceback (most recent call last):
  File "/home/nsindiso/.local/lib/python3.6/site-packages/django/template/base.py", line 835, in _resolve_lookup
    current = current[bit]
KeyError: 'minLength'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/nsindiso/.local/lib/python3.6/site-packages/django/template/base.py", line 843, in _resolve_lookup
    current = getattr(current, bit)
AttributeError: 'dict' object has no attribute 'minLength'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/nsindiso/.local/lib/python3.6/site-packages/django/template/base.py", line 849, in _resolve_lookup
    current = current[int(bit)]
ValueError: invalid literal for int() with base 10: 'minLength'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/nsindiso/.local/lib/python3.6/site-packages/django/template/base.py", line 856, in _resolve_lookup
    (bit, current))  # missing attribute
django.template.base.VariableDoesNotExist: Failed lookup for key [minLength] in {'type': 'string', 'description': 'Verbose description', 'default': 'MDT'}
[2021-04-27 20:55:24.746 DEBUG                    django.template : 878]
Exception while resolving variable 'max' in template 'ysdevices/device_form.html'.
Traceback (most recent call last):
  File "/home/nsindiso/.local/lib/python3.6/site-packages/django/template/base.py", line 835, in _resolve_lookup
    current = current[bit]
KeyError: 'max'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/nsindiso/.local/lib/python3.6/site-packages/django/template/base.py", line 843, in _resolve_lookup
    current = getattr(current, bit)
AttributeError: 'dict' object has no attribute 'max'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/nsindiso/.local/lib/python3.6/site-packages/django/template/base.py", line 849, in _resolve_lookup
    current = current[int(bit)]
ValueError: invalid literal for int() with base 10: 'max'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/nsindiso/.local/lib/python3.6/site-packages/django/template/base.py", line 856, in _resolve_lookup
    (bit, current))  # missing attribute
django.template.base.VariableDoesNotExist: Failed lookup for key [max] in {'type': 'string', 'description': 'Verbose description', 'default': 'MDT'}
[2021-04-27 20:55:24.747 DEBUG                    django.template : 878]
Exception while resolving variable 'maxLength' in template 'ysdevices/device_form.html'.
Traceback (most recent call last):
  File "/home/nsindiso/.local/lib/python3.6/site-packages/django/template/base.py", line 835, in _resolve_lookup
    current = current[bit]
KeyError: 'maxLength'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/nsindiso/.local/lib/python3.6/site-packages/django/template/base.py", line 843, in _resolve_lookup
    current = getattr(current, bit)
AttributeError: 'dict' object has no attribute 'maxLength'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/nsindiso/.local/lib/python3.6/site-packages/django/template/base.py", line 849, in _resolve_lookup
    current = current[int(bit)]
ValueError: invalid literal for int() with base 10: 'maxLength'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/nsindiso/.local/lib/python3.6/site-packages/django/template/base.py", line 856, in _resolve_lookup
    (bit, current))  # missing attribute
django.template.base.VariableDoesNotExist: Failed lookup for key [maxLength] in {'type': 'string', 'description': 'Verbose description', 'default': 'MDT'}
[2021-04-27 20:55:24.748 DEBUG                    django.template : 878]
Exception while resolving variable 'step' in template 'ysdevices/device_form.html'.
Traceback (most recent call last):
  File "/home/nsindiso/.local/lib/python3.6/site-packages/django/template/base.py", line 835, in _resolve_lookup
    current = current[bit]
KeyError: 'step'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/nsindiso/.local/lib/python3.6/site-packages/django/template/base.py", line 843, in _resolve_lookup
    current = getattr(current, bit)
AttributeError: 'dict' object has no attribute 'step'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/nsindiso/.local/lib/python3.6/site-packages/django/template/base.py", line 849, in _resolve_lookup
    current = current[int(bit)]
ValueError: invalid literal for int() with base 10: 'step'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/nsindiso/.local/lib/python3.6/site-packages/django/template/base.py", line 856, in _resolve_lookup
    (bit, current))  # missing attribute
django.template.base.VariableDoesNotExist: Failed lookup for key [step] in {'type': 'string', 'description': 'Verbose description', 'default': 'MDT'}
[2021-04-27 20:55:24.749 DEBUG                    django.template : 878]
Exception while resolving variable 'required' in template 'ysdevices/device_form.html'.
Traceback (most recent call last):
  File "/home/nsindiso/.local/lib/python3.6/site-packages/django/template/base.py", line 835, in _resolve_lookup
    current = current[bit]
KeyError: 'required'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/nsindiso/.local/lib/python3.6/site-packages/django/template/base.py", line 843, in _resolve_lookup
    current = getattr(current, bit)
AttributeError: 'dict' object has no attribute 'required'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/nsindiso/.local/lib/python3.6/site-packages/django/template/base.py", line 849, in _resolve_lookup
    current = current[int(bit)]
ValueError: invalid literal for int() with base 10: 'required'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/nsindiso/.local/lib/python3.6/site-packages/django/template/base.py", line 856, in _resolve_lookup
    (bit, current))  # missing attribute
django.template.base.VariableDoesNotExist: Failed lookup for key [required] in {'type': 'string', 'description': 'Verbose description', 'default': 'MDT'}
[2021-04-27 20:55:24.750 DEBUG                    django.template : 878]
Exception while resolving variable 'label' in template 'ysdevices/device_form.html'.
Traceback (most recent call last):
  File "/home/nsindiso/.local/lib/python3.6/site-packages/django/template/base.py", line 835, in _resolve_lookup
    current = current[bit]
KeyError: 'label'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/nsindiso/.local/lib/python3.6/site-packages/django/template/base.py", line 843, in _resolve_lookup
    current = getattr(current, bit)
AttributeError: 'dict' object has no attribute 'label'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/nsindiso/.local/lib/python3.6/site-packages/django/template/base.py", line 849, in _resolve_lookup
    current = current[int(bit)]
ValueError: invalid literal for int() with base 10: 'label'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/nsindiso/.local/lib/python3.6/site-packages/django/template/base.py", line 856, in _resolve_lookup
    (bit, current))  # missing attribute
django.template.base.VariableDoesNotExist: Failed lookup for key [label] in {'type': 'string', 'description': 'Verbose description', 'default': 'MDT'}
[2021-04-27 20:55:24.752 DEBUG                    django.template : 878]
Exception while resolving variable 'required' in template 'ysdevices/device_form.html'.
Traceback (most recent call last):
  File "/home/nsindiso/.local/lib/python3.6/site-packages/django/template/base.py", line 835, in _resolve_lookup
    current = current[bit]
KeyError: 'required'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/nsindiso/.local/lib/python3.6/site-packages/django/template/base.py", line 843, in _resolve_lookup
    current = getattr(current, bit)
AttributeError: 'dict' object has no attribute 'required'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/nsindiso/.local/lib/python3.6/site-packages/django/template/base.py", line 849, in _resolve_lookup
    current = current[int(bit)]
ValueError: invalid literal for int() with base 10: 'required'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/nsindiso/.local/lib/python3.6/site-packages/django/template/base.py", line 856, in _resolve_lookup
    (bit, current))  # missing attribute
django.template.base.VariableDoesNotExist: Failed lookup for key [required] in {'type': 'string', 'description': 'Verbose description', 'default': 'MDT'}
[2021-04-27 20:55:24.753 DEBUG                    django.template : 878]
Exception while resolving variable 'min' in template 'ysdevices/device_form.html'.
Traceback (most recent call last):
  File "/home/nsindiso/.local/lib/python3.6/site-packages/django/template/base.py", line 835, in _resolve_lookup
    current = current[bit]
KeyError: 'min'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/nsindiso/.local/lib/python3.6/site-packages/django/template/base.py", line 843, in _resolve_lookup
    current = getattr(current, bit)
AttributeError: 'dict' object has no attribute 'min'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/nsindiso/.local/lib/python3.6/site-packages/django/template/base.py", line 849, in _resolve_lookup
    current = current[int(bit)]
ValueError: invalid literal for int() with base 10: 'min'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/nsindiso/.local/lib/python3.6/site-packages/django/template/base.py", line 856, in _resolve_lookup
    (bit, current))  # missing attribute
django.template.base.VariableDoesNotExist: Failed lookup for key [min] in {'type': 'string', 'description': 'Address or hostname to use as default for all protocols', 'required': True, 'default': '192.168.42.82'}
[2021-04-27 20:55:24.755 DEBUG                    django.template : 878]
Exception while resolving variable 'minLength' in template 'ysdevices/device_form.html'.
Traceback (most recent call last):
  File "/home/nsindiso/.local/lib/python3.6/site-packages/django/template/base.py", line 835, in _resolve_lookup
    current = current[bit]
KeyError: 'minLength'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/nsindiso/.local/lib/python3.6/site-packages/django/template/base.py", line 843, in _resolve_lookup
    current = getattr(current, bit)
AttributeError: 'dict' object has no attribute 'minLength'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/nsindiso/.local/lib/python3.6/site-packages/django/template/base.py", line 849, in _resolve_lookup
    current = current[int(bit)]
ValueError: invalid literal for int() with base 10: 'minLength'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/nsindiso/.local/lib/python3.6/site-packages/django/template/base.py", line 856, in _resolve_lookup
    (bit, current))  # missing attribute
django.template.base.VariableDoesNotExist: Failed lookup for key [minLength] in {'type': 'string', 'description': 'Address or hostname to use as default for all protocols', 'required': True, 'default': '192.168.42.82'}
[2021-04-27 20:55:24.755 DEBUG                    django.template : 878]
Exception while resolving variable 'max' in template 'ysdevices/device_form.html'.
Traceback (most recent call last):
  File "/home/nsindiso/.local/lib/python3.6/site-packages/django/template/base.py", line 835, in _resolve_lookup
    current = current[bit]
KeyError: 'max'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/nsindiso/.local/lib/python3.6/site-packages/django/template/base.py", line 843, in _resolve_lookup
    current = getattr(current, bit)
AttributeError: 'dict' object has no attribute 'max'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/nsindiso/.local/lib/python3.6/site-packages/django/template/base.py", line 849, in _resolve_lookup
    current = current[int(bit)]
ValueError: invalid literal for int() with base 10: 'max'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/nsindiso/.local/lib/python3.6/site-packages/django/template/base.py", line 856, in _resolve_lookup
    (bit, current))  # missing attribute
django.template.base.VariableDoesNotExist: Failed lookup for key [max] in {'type': 'string', 'description': 'Address or hostname to use as default for all protocols', 'required': True, 'default': '192.168.42.82'}
[2021-04-27 20:55:24.756 DEBUG                    django.template : 878]
Exception while resolving variable 'maxLength' in template 'ysdevices/device_form.html'.
Traceback (most recent call last):
  File "/home/nsindiso/.local/lib/python3.6/site-packages/django/template/base.py", line 835, in _resolve_lookup
    current = current[bit]
KeyError: 'maxLength'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/nsindiso/.local/lib/python3.6/site-packages/django/template/base.py", line 843, in _resolve_lookup
    current = getattr(current, bit)
AttributeError: 'dict' object has no attribute 'maxLength'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/nsindiso/.local/lib/python3.6/site-packages/django/template/base.py", line 849, in _resolve_lookup
    current = current[int(bit)]
ValueError: invalid literal for int() with base 10: 'maxLength'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/nsindiso/.local/lib/python3.6/site-packages/django/template/base.py", line 856, in _resolve_lookup
    (bit, current))  # missing attribute
django.template.base.VariableDoesNotExist: Failed lookup for key [maxLength] in {'type': 'string', 'description': 'Address or hostname to use as default for all protocols', 'required': True, 'default': '192.168.42.82'}
[2021-04-27 20:55:24.758 DEBUG                    django.template : 878]
Exception while resolving variable 'step' in template 'ysdevices/device_form.html'.
Traceback (most recent call last):
  File "/home/nsindiso/.local/lib/python3.6/site-packages/django/template/base.py", line 835, in _resolve_lookup
    current = current[bit]
KeyError: 'step'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/nsindiso/.local/lib/python3.6/site-packages/django/template/base.py", line 843, in _resolve_lookup
    current = getattr(current, bit)
AttributeError: 'dict' object has no attribute 'step'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/nsindiso/.local/lib/python3.6/site-packages/django/template/base.py", line 849, in _resolve_lookup
    current = current[int(bit)]
ValueError: invalid literal for int() with base 10: 'step'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/nsindiso/.local/lib/python3.6/site-packages/django/template/base.py", line 856, in _resolve_lookup
    (bit, current))  # missing attribute
django.template.base.VariableDoesNotExist: Failed lookup for key [step] in {'type': 'string', 'description': 'Address or hostname to use as default for all protocols', 'required': True, 'default': '192.168.42.82'}
[2021-04-27 20:55:24.759 DEBUG                    django.template : 878]
Exception while resolving variable 'label' in template 'ysdevices/device_form.html'.
Traceback (most recent call last):
  File "/home/nsindiso/.local/lib/python3.6/site-packages/django/template/base.py", line 835, in _resolve_lookup
    current = current[bit]
KeyError: 'label'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/nsindiso/.local/lib/python3.6/site-packages/django/template/base.py", line 843, in _resolve_lookup
    current = getattr(current, bit)
AttributeError: 'dict' object has no attribute 'label'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/nsindiso/.local/lib/python3.6/site-packages/django/template/base.py", line 849, in _resolve_lookup
    current = current[int(bit)]
ValueError: invalid literal for int() with base 10: 'label'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/nsindiso/.local/lib/python3.6/site-packages/django/template/base.py", line 856, in _resolve_lookup
    (bit, current))  # missing attribute
django.template.base.VariableDoesNotExist: Failed lookup for key [label] in {'type': 'string', 'description': 'Address or hostname to use as default for all protocols', 'required': True, 'default': '192.168.42.82'}
[2021-04-27 20:55:24.761 DEBUG                    django.template : 878]
Exception while resolving variable 'min' in template 'ysdevices/device_form.html'.
Traceback (most recent call last):
  File "/home/nsindiso/.local/lib/python3.6/site-packages/django/template/base.py", line 835, in _resolve_lookup
    current = current[bit]
KeyError: 'min'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/nsindiso/.local/lib/python3.6/site-packages/django/template/base.py", line 843, in _resolve_lookup
    current = getattr(current, bit)
AttributeError: 'dict' object has no attribute 'min'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/nsindiso/.local/lib/python3.6/site-packages/django/template/base.py", line 849, in _resolve_lookup
    current = current[int(bit)]
ValueError: invalid literal for int() with base 10: 'min'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/nsindiso/.local/lib/python3.6/site-packages/django/template/base.py", line 856, in _resolve_lookup
    (bit, current))  # missing attribute
django.template.base.VariableDoesNotExist: Failed lookup for key [min] in {'type': 'string', 'description': 'Username used as default for all protocols', 'maxLength': 50, 'default': 'Nsindiso'}
[2021-04-27 20:55:24.762 DEBUG                    django.template : 878]
Exception while resolving variable 'minLength' in template 'ysdevices/device_form.html'.
Traceback (most recent call last):
  File "/home/nsindiso/.local/lib/python3.6/site-packages/django/template/base.py", line 835, in _resolve_lookup
    current = current[bit]
KeyError: 'minLength'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/nsindiso/.local/lib/python3.6/site-packages/django/template/base.py", line 843, in _resolve_lookup
    current = getattr(current, bit)
AttributeError: 'dict' object has no attribute 'minLength'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/nsindiso/.local/lib/python3.6/site-packages/django/template/base.py", line 849, in _resolve_lookup
    current = current[int(bit)]
ValueError: invalid literal for int() with base 10: 'minLength'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/nsindiso/.local/lib/python3.6/site-packages/django/template/base.py", line 856, in _resolve_lookup
    (bit, current))  # missing attribute
django.template.base.VariableDoesNotExist: Failed lookup for key [minLength] in {'type': 'string', 'description': 'Username used as default for all protocols', 'maxLength': 50, 'default': 'Nsindiso'}
[2021-04-27 20:55:24.763 DEBUG                    django.template : 878]
Exception while resolving variable 'max' in template 'ysdevices/device_form.html'.
Traceback (most recent call last):
  File "/home/nsindiso/.local/lib/python3.6/site-packages/django/template/base.py", line 835, in _resolve_lookup
    current = current[bit]
KeyError: 'max'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/nsindiso/.local/lib/python3.6/site-packages/django/template/base.py", line 843, in _resolve_lookup
    current = getattr(current, bit)
AttributeError: 'dict' object has no attribute 'max'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/nsindiso/.local/lib/python3.6/site-packages/django/template/base.py", line 849, in _resolve_lookup
    current = current[int(bit)]
ValueError: invalid literal for int() with base 10: 'max'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/nsindiso/.local/lib/python3.6/site-packages/django/template/base.py", line 856, in _resolve_lookup
    (bit, current))  # missing attribute
django.template.base.VariableDoesNotExist: Failed lookup for key [max] in {'type': 'string', 'description': 'Username used as default for all protocols', 'maxLength': 50, 'default': 'Nsindiso'}
[2021-04-27 20:55:24.764 DEBUG                    django.template : 878]
Exception while resolving variable 'step' in template 'ysdevices/device_form.html'.
Traceback (most recent call last):
  File "/home/nsindiso/.local/lib/python3.6/site-packages/django/template/base.py", line 835, in _resolve_lookup
    current = current[bit]
KeyError: 'step'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/nsindiso/.local/lib/python3.6/site-packages/django/template/base.py", line 843, in _resolve_lookup
    current = getattr(current, bit)
AttributeError: 'dict' object has no attribute 'step'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/nsindiso/.local/lib/python3.6/site-packages/django/template/base.py", line 849, in _resolve_lookup
    current = current[int(bit)]
ValueError: invalid literal for int() with base 10: 'step'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/nsindiso/.local/lib/python3.6/site-packages/django/template/base.py", line 856, in _resolve_lookup
    (bit, current))  # missing attribute
django.template.base.VariableDoesNotExist: Failed lookup for key [step] in {'type': 'string', 'description': 'Username used as default for all protocols', 'maxLength': 50, 'default': 'Nsindiso'}
[2021-04-27 20:55:24.765 DEBUG                    django.template : 878]
Exception while resolving variable 'required' in template 'ysdevices/device_form.html'.
Traceback (most recent call last):
  File "/home/nsindiso/.local/lib/python3.6/site-packages/django/template/base.py", line 835, in _resolve_lookup
    current = current[bit]
KeyError: 'required'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/nsindiso/.local/lib/python3.6/site-packages/django/template/base.py", line 843, in _resolve_lookup
    current = getattr(current, bit)
AttributeError: 'dict' object has no attribute 'required'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/nsindiso/.local/lib/python3.6/site-packages/django/template/base.py", line 849, in _resolve_lookup
    current = current[int(bit)]
ValueError: invalid literal for int() with base 10: 'required'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/nsindiso/.local/lib/python3.6/site-packages/django/template/base.py", line 856, in _resolve_lookup
    (bit, current))  # missing attribute
django.template.base.VariableDoesNotExist: Failed lookup for key [required] in {'type': 'string', 'description': 'Username used as default for all protocols', 'maxLength': 50, 'default': 'Nsindiso'}
[2021-04-27 20:55:24.766 DEBUG                    django.template : 878]
Exception while resolving variable 'label' in template 'ysdevices/device_form.html'.
Traceback (most recent call last):
  File "/home/nsindiso/.local/lib/python3.6/site-packages/django/template/base.py", line 835, in _resolve_lookup
    current = current[bit]
KeyError: 'label'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/nsindiso/.local/lib/python3.6/site-packages/django/template/base.py", line 843, in _resolve_lookup
    current = getattr(current, bit)
AttributeError: 'dict' object has no attribute 'label'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/nsindiso/.local/lib/python3.6/site-packages/django/template/base.py", line 849, in _resolve_lookup
    current = current[int(bit)]
ValueError: invalid literal for int() with base 10: 'label'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/nsindiso/.local/lib/python3.6/site-packages/django/template/base.py", line 856, in _resolve_lookup
    (bit, current))  # missing attribute
django.template.base.VariableDoesNotExist: Failed lookup for key [label] in {'type': 'string', 'description': 'Username used as default for all protocols', 'maxLength': 50, 'default': 'Nsindiso'}
[2021-04-27 20:55:24.767 DEBUG                    django.template : 878]
Exception while resolving variable 'required' in template 'ysdevices/device_form.html'.
Traceback (most recent call last):
  File "/home/nsindiso/.local/lib/python3.6/site-packages/django/template/base.py", line 835, in _resolve_lookup
    current = current[bit]
KeyError: 'required'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/nsindiso/.local/lib/python3.6/site-packages/django/template/base.py", line 843, in _resolve_lookup
    current = getattr(current, bit)
AttributeError: 'dict' object has no attribute 'required'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/nsindiso/.local/lib/python3.6/site-packages/django/template/base.py", line 849, in _resolve_lookup
    current = current[int(bit)]
ValueError: invalid literal for int() with base 10: 'required'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/nsindiso/.local/lib/python3.6/site-packages/django/template/base.py", line 856, in _resolve_lookup
    (bit, current))  # missing attribute
django.template.base.VariableDoesNotExist: Failed lookup for key [required] in {'type': 'string', 'description': 'Username used as default for all protocols', 'maxLength': 50, 'default': 'Nsindiso'}
[2021-04-27 20:55:24.768 DEBUG                    django.template : 878]
Exception while resolving variable 'min' in template 'ysdevices/device_form.html'.
Traceback (most recent call last):
  File "/home/nsindiso/.local/lib/python3.6/site-packages/django/template/base.py", line 835, in _resolve_lookup
    current = current[bit]
KeyError: 'min'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/nsindiso/.local/lib/python3.6/site-packages/django/template/base.py", line 843, in _resolve_lookup
    current = getattr(current, bit)
AttributeError: 'dict' object has no attribute 'min'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/nsindiso/.local/lib/python3.6/site-packages/django/template/base.py", line 849, in _resolve_lookup
    current = current[int(bit)]
ValueError: invalid literal for int() with base 10: 'min'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/nsindiso/.local/lib/python3.6/site-packages/django/template/base.py", line 856, in _resolve_lookup
    (bit, current))  # missing attribute
django.template.base.VariableDoesNotExist: Failed lookup for key [min] in {'type': 'password', 'description': 'Password used as default for all protocols', 'maxLength': 50, 'default': '_placeholder_'}
[2021-04-27 20:55:24.770 DEBUG                    django.template : 878]
Exception while resolving variable 'minLength' in template 'ysdevices/device_form.html'.
Traceback (most recent call last):
  File "/home/nsindiso/.local/lib/python3.6/site-packages/django/template/base.py", line 835, in _resolve_lookup
    current = current[bit]
KeyError: 'minLength'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/nsindiso/.local/lib/python3.6/site-packages/django/template/base.py", line 843, in _resolve_lookup
    current = getattr(current, bit)
AttributeError: 'dict' object has no attribute 'minLength'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/nsindiso/.local/lib/python3.6/site-packages/django/template/base.py", line 849, in _resolve_lookup
    current = current[int(bit)]
ValueError: invalid literal for int() with base 10: 'minLength'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/nsindiso/.local/lib/python3.6/site-packages/django/template/base.py", line 856, in _resolve_lookup
    (bit, current))  # missing attribute
django.template.base.VariableDoesNotExist: Failed lookup for key [minLength] in {'type': 'password', 'description': 'Password used as default for all protocols', 'maxLength': 50, 'default': '_placeholder_'}
[2021-04-27 20:55:24.772 DEBUG                    django.template : 878]
Exception while resolving variable 'max' in template 'ysdevices/device_form.html'.
Traceback (most recent call last):
  File "/home/nsindiso/.local/lib/python3.6/site-packages/django/template/base.py", line 835, in _resolve_lookup
    current = current[bit]
KeyError: 'max'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/nsindiso/.local/lib/python3.6/site-packages/django/template/base.py", line 843, in _resolve_lookup
    current = getattr(current, bit)
AttributeError: 'dict' object has no attribute 'max'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/nsindiso/.local/lib/python3.6/site-packages/django/template/base.py", line 849, in _resolve_lookup
    current = current[int(bit)]
ValueError: invalid literal for int() with base 10: 'max'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/nsindiso/.local/lib/python3.6/site-packages/django/template/base.py", line 856, in _resolve_lookup
    (bit, current))  # missing attribute
django.template.base.VariableDoesNotExist: Failed lookup for key [max] in {'type': 'password', 'description': 'Password used as default for all protocols', 'maxLength': 50, 'default': '_placeholder_'}
